<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.4.555">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="Chloé R. Nater, Francesco Frassinelli, James A. Martin, Erlend B. Nilsen">

<title>Large-scale spatiotemporal variation in vital rates and population dynamics of an alpine bird</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
  margin-bottom: 0em;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>


<script src="Manuscript_Nater_etal_files/libs/clipboard/clipboard.min.js"></script>
<script src="Manuscript_Nater_etal_files/libs/quarto-html/quarto.js"></script>
<script src="Manuscript_Nater_etal_files/libs/quarto-html/popper.min.js"></script>
<script src="Manuscript_Nater_etal_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="Manuscript_Nater_etal_files/libs/quarto-html/anchor.min.js"></script>
<link href="Manuscript_Nater_etal_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="Manuscript_Nater_etal_files/libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="Manuscript_Nater_etal_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="Manuscript_Nater_etal_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="Manuscript_Nater_etal_files/libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">

  <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
  <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml-full.js" type="text/javascript"></script>

<script type="text/javascript">
const typesetMath = (el) => {
  if (window.MathJax) {
    // MathJax Typeset
    window.MathJax.typeset([el]);
  } else if (window.katex) {
    // KaTeX Render
    var mathElements = el.getElementsByClassName("math");
    var macros = [];
    for (var i = 0; i < mathElements.length; i++) {
      var texText = mathElements[i].firstChild;
      if (mathElements[i].tagName == "SPAN") {
        window.katex.render(texText.data, mathElements[i], {
          displayMode: mathElements[i].classList.contains('display'),
          throwOnError: false,
          macros: macros,
          fleqn: false
        });
      }
    }
  }
}
window.Quarto = {
  typesetMath
};
</script>

</head>

<body class="fullcontent">

<div id="quarto-content" class="page-columns page-rows-contents page-layout-article">

<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Large-scale spatiotemporal variation in vital rates and population dynamics of an alpine bird</h1>
</div>



<div class="quarto-title-meta">

    <div>
    <div class="quarto-title-meta-heading">Author</div>
    <div class="quarto-title-meta-contents">
             <p>Chloé R. Nater, Francesco Frassinelli, James A. Martin, Erlend B. Nilsen </p>
          </div>
  </div>
    
  
    
  </div>
  


</header>


<section id="abstract" class="level1">
<h1>Abstract</h1>
<p>Quantifying temporal and spatial variation in animal population size and demography is a central theme in ecological research and important for directing management and policy. However, this requires field sampling at large spatial extents and over long periods of time, which is not only prohibitively costly but often politically untenable. Participatory monitoring programs (also called citizen science programmes) can alleviate these constraints by recruiting stakeholders and the public to increase the spatial and temporal resolution of sampling effort and hence resulting data. While the majority of participatory monitoring programs are limited by opportunistic sampling designs, we are starting to see the emergence of structured citizen science programs that employ trained volunteers to collect data according to standardized protocols. Simultaneously, there is much ongoing development of statistical models that are increasingly more powerful and able to make more efficient use of field data. Integrated population models (IPMs), for example, are able to use multiple streams of data from different field monitoring programmes and/or multiple aspects of single datasets to estimate population sizes and key vital rates. Here, we developed a multi-area version of a recently developed integrated distance sampling model (IDSM) and applied it to data from a large-scale participatory monitoring program – the “Hønsefuglportalen” – to study spatio-temporal variation in population dynamics of willow ptarmigan (<em>Lagopus lagopus</em>) in Norway. We constructed an open and reproducible workflow for exploring temporal, spatial (latitudinal, longitudinal, altitudinal), and residual variation in recruitment, survival, and population density, as well as relationships between vital rates and relevant covariates and signals of density dependence. Recruitment rates varied more across space than over time, while the opposite was the case for survival. Slower life history patterns (higher survival, lower recruitment) appeared to be more common at higher latitudes and altitudes, portending differential effects of climate change on ptarmigan across their range. While there was variation in the magnitude of the effect small rodent occupancy had on recruitment, the relationships were predominantly positive and thus consistent with the alternative prey hypothesis. Notably, the accurate estimation of covariate effect was only made possible by integrating data from several monitoring areas for analysis. Our study highlights the potential of participatory monitoring and integrated modelling approaches for estimating and understanding spatio-temporal patterns in species abundance and demographic rates, and showcases how corresponding workflows can be set up in reproducible and semi-automated ways that increase their usefulness for informing management and regular reporting towards national and international biodiversity frameworks.</p>
</section>
<section id="introduction" class="level1">
<h1>Introduction</h1>
<p>There is growing demand for biodiversity indicators from international unions, national governments, local management bodies, and corporate and industry actors. Indicators should ideally represent a wide range of biodiversity’s states and functions <span class="citation" data-cites="pereira2013 jetz2019">(e.g.&nbsp;Essential Biodiversity Variables, <a href="#ref-pereira2013" role="doc-biblioref">Pereira et al. 2013</a>; <a href="#ref-jetz2019" role="doc-biblioref">Jetz et al. 2019</a>)</span>, yet the development of suitable indicators for certain attributes, such as species abundance and demography, has been more difficult than for others <span class="citation" data-cites="schmeller2018 waldock2022">(<a href="#ref-schmeller2018" role="doc-biblioref">Schmeller et al. 2018</a>; <a href="#ref-waldock2022" role="doc-biblioref">Waldock et al. 2022</a>)</span>. This is at least partially due to challenging requirements regarding spatial scales of useful biodiversity indicators. On one hand, indicators need to be representative at large geographic scales, for example, in the context of countries’ reporting towards biodiversity targets <span class="citation" data-cites="feld2009">(e.g. <a href="#ref-feld2009" role="doc-biblioref">Feld et al. 2009</a>)</span>. On the other hand, indicators also ideally have good spatial resolution, as the scales relevant for local-level management and planning are often much smaller <span class="citation" data-cites="stevenson2021">(<a href="#ref-stevenson2021" role="doc-biblioref">Stevenson et al. 2021</a>)</span>. This latter requirement is particularly crucial for infrastructure development strategies and for species management and conservation, both of which tend to require knowledge on species abundance and population dynamics (i.e.&nbsp;demographic rates) that is relevant for county- or municipality-level decision making <span class="citation" data-cites="christie2020">(<a href="#ref-christie2020" role="doc-biblioref">Christie et al. 2020</a>)</span>. Another reason why abundance and population indicators ideally come with good spatial resolution is that there can be substantial amounts of variation in population dynamics and life history of species across space <span class="citation" data-cites="robinson2014 horswill2019">(e.g. <a href="#ref-robinson2014" role="doc-biblioref">Robinson, Morrison, and Baillie 2014</a>; <a href="#ref-horswill2019" role="doc-biblioref">Horswill et al. 2019</a>)</span>. Such variation can arise from differences in ecological contexts (including local habitat and weather conditions, hunting pressures, and interspecific interactions, e.g. <span class="citation" data-cites="nilsen2009">Erlend B. Nilsen et al. (<a href="#ref-nilsen2009" role="doc-biblioref">2009</a>)</span>; <span class="citation" data-cites="bond2021">Bond et al. (<a href="#ref-bond2021" role="doc-biblioref">2021</a>)</span>) and needs to be accounted for when developing sustainable strategies for area use, harvest management, and species and biodiversity conservation <span class="citation" data-cites="williams2002">(<a href="#ref-williams2002" role="doc-biblioref">Williams, Nichols, and Conroy 2002</a>)</span>.</p>
<p>While large-scale, spatially-explicit indicators for species abundance and populations are clearly needed, development and practical implementation are greatly limited due to the reliance of such indicators on the availability of data from large-scale, long-term monitoring programmes <span class="citation" data-cites="proencca2017">(<a href="#ref-proencca2017" role="doc-biblioref">Proença et al. 2017</a>)</span>. Consequently, many countries have been working on setting up, maintaining, and improving such monitoring programmes over the last few decades. Many now well-established programmes focus on breeding birds and butterflies, and examples include the North American Breeding Bird Survey (<a href="https://www.pwrc.usgs.gov/bbs/" class="uri">https://www.pwrc.usgs.gov/bbs/</a>), the PanEuropean Common Bird Monitoring Scheme (<a href="https://pecbms.info/" class="uri">https://pecbms.info/</a>), the UK Butterfly Monitoring Scheme (<a href="https://ukbms.org/" class="uri">https://ukbms.org/</a>), the Game and Wildlife Conservation Trust Partridge Count Scheme <span class="citation" data-cites="aebischer2010">(<a href="#ref-aebischer2010" role="doc-biblioref">Aebischer and Ewald 2010</a>)</span>, and the Swiss Biodiversity Monitoring program (<a href="https://www.biodiversitymonitoring.ch/" class="uri">https://www.biodiversitymonitoring.ch/</a>).</p>
<p>There is a natural trade-off between quality and quantity of data that can be collected in any monitoring programme: collecting high quality data in a structured manner is costly, requires trained specialists, and hinges on a sufficient degree of top-down control of the programme. This often limits the amount of data that can be collected, while participatory monitoring, i.e.&nbsp;the collection of ecological data by members of the public <span class="citation" data-cites="fraisl2022">(also called citizen or community science, <a href="#ref-fraisl2022" role="doc-biblioref">Fraisl et al. 2022</a>)</span>, allows to greatly reduce costs and extend spatial and taxonomic scales of monitoring at the expense of data quality and risk of bias <span class="citation" data-cites="johnston2023">(<a href="#ref-johnston2023" role="doc-biblioref">Johnston, Matechou, and Dennis 2023</a>)</span>. Consequently, many large-scale monitoring programmes are often limited to presence(-absence) or very simple count observations, making them suitable for the development of indicators of species distributions and perhaps population trends, but usually not of abundance, population dynamics, and demographic rates <span class="citation" data-cites="dickinson2010 johnston2023">(<a href="#ref-dickinson2010" role="doc-biblioref">Dickinson, Zuckerberg, and Bonter 2010</a>; <a href="#ref-johnston2023" role="doc-biblioref">Johnston, Matechou, and Dennis 2023</a>)</span>. The exception here are monitoring programmes that succeed in making use of a large number of volunteers that have been trained to collect data and record metadata in a structured manner and according to a carefully designed protocol. For example, in the United States hunters participate in the collection of bands and wings from harvested American Woodcock (<em>Scopolax minor)</em> to estimate survival and age ratios <span class="citation" data-cites="zimmerman2010">(<a href="#ref-zimmerman2010" role="doc-biblioref">Zimmerman et al. 2010</a>)</span>. At the European level, the recently established initiative “European Observatory of Wildlife” is offering common field- and analyses protocols and aims to establish a network of “observation points” for monitoring wildlife populations at the European level (<a href="https://wildlifeobservatory.org/" class="uri">https://wildlifeobservatory.org/</a>). In Norway there is a monitoring programme for terrestrial game bird species called “Hønsefuglportalen” (= “game bird portal”, <a href="https://honsefugl.nina.no/Innsyn/en" class="uri">https://honsefugl.nina.no/Innsyn/en</a>). It is a line transect survey programme carried out annually in &gt;120 localities across the country (&gt;2000 transects) by trained volunteers using pointing dogs. The programme has a well developed protocol for recording bird observations, auxiliary data, and relevant metadata and established routines for quality control and annual releases of publicly available data via the Global Biodiversity Information Facility (GBIF). As such, it is particularly well suited to become part of a workflow for producing and updating abundance and population indicators on an annual basis.</p>
<p>The line transect data from “Hønsefuglportalen” has been used previously for estimating abundance trends of willow ptarmigan (<em>Lagopus lagopus</em>) across Norway <span class="citation" data-cites="bowler2020 nilsen2020">(e.g. <a href="#ref-bowler2020" role="doc-biblioref">Bowler et al. 2020</a>; <a href="#ref-nilsen2020" role="doc-biblioref">E. B. Nilsen and Rød-Eriksen 2020</a>)</span>, and to test a range of relevant ecological hypotheses <span class="citation" data-cites="bowler2020 Breisjøberget_etal_2018">(<a href="#ref-bowler2020" role="doc-biblioref">Bowler et al. 2020</a>; <a href="#ref-Breisjøberget_etal_2018" role="doc-biblioref">Breisjøberget, Odden, Wegge, et al. 2018</a>)</span>. However, large-scale estimation of demographic rates underlying abundance trends has thus far remained an untapped potential of the dataset. <span class="citation" data-cites="nilsen2023">E. B. Nilsen and Nater (<a href="#ref-nilsen2023" role="doc-biblioref">2024</a>)</span> recently developed a novel integrated distance sampling model (IDSM) which successfully uses the age of individuals detected along line transects data coupled with radio-telemetry data to jointly estimate abundance, survival, and recruitment across years. In this study, we adapt and extend the model of <span class="citation" data-cites="nilsen2023">E. B. Nilsen and Nater (<a href="#ref-nilsen2023" role="doc-biblioref">2024</a>)</span> to run not just on a single site but on all areas with publicly available line transect data from “Hønsefuglportalen” simultaneously. Unlike several previous studies applying integrated models for population dynamics to multiple (sub-) populations separately and comparing results <span class="citation" data-cites="robinson2014 nater2023">(e.g. <a href="#ref-robinson2014" role="doc-biblioref">Robinson, Morrison, and Baillie 2014</a>; <a href="#ref-nater2023" role="doc-biblioref">Chloé R. Nater et al. 2023</a>)</span>, we opt for an approach explicitly integrating across space, thus allowing for sharing of information across locations and – in effect – space-for-time substitution <span class="citation" data-cites="horswill2019 morrison2022">(e.g. <a href="#ref-horswill2019" role="doc-biblioref">Horswill et al. 2019</a>; <a href="#ref-morrison2022" role="doc-biblioref">Morrison et al. 2022</a>)</span>. We then apply the resulting multi-area IDSM to “Hønsefuglportalen” data on willow ptarmigan to estimate population size, age-structure, survival, recruitment, and impacts of small rodent occupancy across 41 reporting districts and 15 years (2007-2021) for this culturally important small-game species. We further embed the modelling workflow in reproducible, semi-automated pipelines that will greatly facilitate the repeated calculation of abundance and population indicators at different spatial scales as new data are added every year.</p>
</section>
<section id="methods" class="level1">
<h1>Methods</h1>
<section id="study-species" class="level2">
<h2 class="anchored" data-anchor-id="study-species">Study species</h2>
<p>The willow ptarmigan is a tetraoid bird with a circumpolar distribution, mainly inhabiting sub-alpine and arctic ecosystems <span class="citation" data-cites="fuglei2020">(see e.g. <a href="#ref-fuglei2020" role="doc-biblioref">Fuglei et al. 2020</a>)</span>. While the species is currently listed as Least Concern (LC) both in the global and Norwegian Red List of Species, it has undergone rather dramatic declines in abundance in Norway since the turn of the 20th century <span class="citation" data-cites="hjeljord2022">(<a href="#ref-hjeljord2022" role="doc-biblioref">Hjeljord and Loe 2022</a>)</span>. The main reasons for the long-term decline in abundance remain unresolved, but the willow ptarmigan is considered a sentinel species that is sensitive to both climate change and land use changes <span class="citation" data-cites="Henden2017 Storch2007">(<a href="#ref-Henden2017" role="doc-biblioref">John-André Henden et al. 2017</a>; <a href="#ref-Storch2007" role="doc-biblioref">Storch 2007</a>)</span>. Moreover, being one of only a handful of bird species that spend the winter in mountain ecosystems in Scandinavia, they are important components of the ecosystem as prey for resident predators, such as the gyrfalcon <span class="citation" data-cites="Alastair2020">(<a href="#ref-Alastair2020" role="doc-biblioref">Franke et al. 2020</a>)</span>. The willow ptarmigan has a relatively fast pace of life <span class="citation" data-cites="sandercock2005 Steen1996">(<a href="#ref-sandercock2005" role="doc-biblioref">Sandercock, Martin, and Hannon 2005</a>; <a href="#ref-Steen1996" role="doc-biblioref">Steen H. and Erikstad 1996</a>)</span>, and can display substantial spatio-temporal variation in demographic rates <span class="citation" data-cites="bowler2020">(<a href="#ref-bowler2020" role="doc-biblioref">Bowler et al. 2020</a>)</span>. Population dynamics are characterized by large inter-annual fluctuations in abundance <span class="citation" data-cites="hjeljord2022">(<a href="#ref-hjeljord2022" role="doc-biblioref">Hjeljord and Loe 2022</a>)</span>, and previous research has tied these fluctuations to rodent cycles through shared predators <span class="citation" data-cites="hagen1952 bowler2020">(<a href="#ref-hagen1952" role="doc-biblioref">Hagen 1952</a>; <a href="#ref-bowler2020" role="doc-biblioref">Bowler et al. 2020</a>)</span>. This tight relationship between the breeding success of ground nesting birds and the rodent cycle is known as the Alternative Prey Hypothesis (APH) and has been a central part of research on Fennoscandian grouse population dynamics for many decades <span class="citation" data-cites="Elton1942 hagen1952 Linden1988 Steen1988">(<a href="#ref-Elton1942" role="doc-biblioref">Elton 1942</a>; <a href="#ref-hagen1952" role="doc-biblioref">Hagen 1952</a>; <a href="#ref-Linden1988" role="doc-biblioref">Linden 1988</a>; <a href="#ref-Steen1988" role="doc-biblioref">J. B. Steen et al. 1988</a>)</span>. In addition, spring weather conditions and phenology is known to have considerable effects on breeding success and recruitment rates <span class="citation" data-cites="Eriksen2023 Steen1988">(<a href="#ref-Eriksen2023" role="doc-biblioref">Eriksen et al. 2023</a>; <a href="#ref-Steen1988" role="doc-biblioref">J. B. Steen et al. 1988</a>)</span>. Across their distributional range, willow ptarmigan are an iconic species with a high cultural value, partly linked to their popularity as game species. The latter means that information about spatio-temporal variation in demographic rates and population dynamics is particularly important in order to design sustainable harvest strategies <span class="citation" data-cites="Eriksen2023">(<a href="#ref-Eriksen2023" role="doc-biblioref">Eriksen et al. 2023</a>)</span>. In addition, being a sentinel species, the willow ptarmigan is well suited as an indicator species for ecosystem status; in Norway it is included in both the main national biodiversity <span class="citation" data-cites="jakobsson2020">(Nature Index for Norway, <a href="#ref-jakobsson2020" role="doc-biblioref">Jakobsson and Pedersen 2020</a>, <a href="https://www.naturindeks.no/Indicators/lirype" class="uri" role="doc-biblioref">https://www.naturindeks.no/Indicators/lirype</a>)</span> and ecosystem condition <span class="citation" data-cites="framstad2022">(Assessment of the Ecological Condition, <a href="#ref-framstad2022" role="doc-biblioref">Framstad et al. 2022</a>)</span> assessments.</p>
</section>
<section id="data-collection-management-and-preparation" class="level2">
<h2 class="anchored" data-anchor-id="data-collection-management-and-preparation">Data collection, management, and preparation</h2>
<section id="line-transect-sampling" class="level3">
<h3 class="anchored" data-anchor-id="line-transect-sampling">Line transect sampling</h3>
<p>The line transect survey data were collected through a structured participatory monitoring program called “Hønsefuglportalen” (<a href="https://honsefugl.nina.no/Innsyn/en" class="uri">https://honsefugl.nina.no/Innsyn/en</a>). In the first three weeks of August each year, trained volunteer fieldworkers collect observations of willow ptarmigan and other grouse species (rock ptarmigan <em>Lagopus muta</em>, black grouse <em>Lyrurus tetrix</em>, and capercaillie <em>Tetrao urogallus</em>) along predefined line transects. To increase the detection probability, fieldworkers use pointing dogs to locate the birds. A survey team typically consists of at least two people (one dog handler and one person responsible for following the transect line) and one dog. Often, more than one dog is used for a survey, but only one dog should be used at a time. The transect lines vary in length, but are typically between 1 and 8 km (range: 0.3-16.2 km, median: 3 km). When birds are observed, the exact location of observation is reported, along with the perpendicular distance from the transect line, as well as the age and sex of the birds. An observation typically includes 1 - 12 birds (mean = 5.6), with groups &gt; 1 typically representing one brood (female and or male with young-of-the-year chicks). When the surveys are conducted in August, the chicks of the year are able to fly but can be distinguished from older birds as they are still of smaller body size. Since 2019, most of the data has been collected using a mobile app tailored to the monitoring programme, where the field workers can register and get access to the transect lines allocated to them by the local organizers. Prior to 2019, data were collected on a dedicated fieldwork form, and entered manually in a web portal afterwards. After field data has been registered, it undergoes several steps of quality control carried out by local stakeholders and personnel from the Norwegian Institute for Nature Research (NINA). Surveys are carried out on both public and private land. After an initial embargo period, all data from public land are published and made freely available as a sampling-event data set on GBIF (<a href="https://www.gbif.org/sampling-event-data" class="uri">https://www.gbif.org/sampling-event-data</a>). The published datasets contain both metadata about the transect surveys (survey date, line transect length and location, study area ID, etc.) and bird observation data (species, number of birds of different categories (adult males, adult females, juveniles, and birds of unknown category), perpendicular distance to transect line, exact location, and time of observation). Formally, the data from public land is published as three distinct data sets, one for each of the main public land administrators (Statskog, FeFo and Fjellstyrene, respectively).</p>
<p>Notably, the program is not designed as a centralized national monitoring programme, but rather a collection of local and regional survey programs. All involved survey areas use a common field protocol and data collection model. In addition, the local study designs are reviewed by staff at NINA, and common recommendations for study design are provided. However, because participation by stakeholders is voluntary, the spatial distribution of transects and sampling effort is not homogeneous across space. In general, sampling effort is higher in South-Eastern and Central Norway, intermediate in Northern Norway, and low in Western and Southern Norway.</p>
<p>In this study we used all publicly available data for the period 2007-2021, which included a total of 2225 transects in 41 different reporting districts spanning 9 counties and 50 municipalities. Transects on which no willow ptarmigan were observed during the study period (i.e.&nbsp;species absence likely due to low habitat suitability) were not included. After this initial filtering, a total of 2077 transects were included in the analyses.</p>
</section>
<section id="radio-telemetry-study-in-lierne" class="level3">
<h3 class="anchored" data-anchor-id="radio-telemetry-study-in-lierne">Radio-telemetry study in Lierne</h3>
<p>The model of <span class="citation" data-cites="nilsen2023">E. B. Nilsen and Nater (<a href="#ref-nilsen2023" role="doc-biblioref">2024</a>)</span> integrated line transect data with radio-telemetry data from an ongoing field study of marked willow ptarmigans in Lierne municipality in Central Norway. From 2015 to 2019, around 50 birds were captured in winter (late February or early March) each year and fitted with VHF collars. This included males and females, and young-of-the year (8-9 months at capture) and adult (&gt;1 year old) birds. The marked birds were then monitored on a regular basis until they either i) died, ii) their transmitter’s battery stopped working, or iii) we lost contact with the bird for other reasons. For most of the year, the birds were monitored at least once a month by radio triangulation. Most of the fieldwork was conducted from the ground, but to avoid data gaps, the birds were also triangulated from helicopters in May, September, and November. During the breeding and chick-rearing season (May to July) birds were monitored more often, and during December and January we obtained fewer observations due to challenging field work conditions. A proportion of the birds were harvested annually in the regular recreational harvest, and birds that were harvested were reported as shot to the field personnel. In addition, as the collars had mortality switches, we were also able to locate and retrieve a high proportion of birds that died from natural causes, resulting in a known-fate mark-recapture dataset. Several previous studies found no effect of the telemetry devices on ptarmigan survival and further details on the radio-telemetry study can be found in <span class="citation" data-cites="Israelsen2020">Israelsen et al. (<a href="#ref-Israelsen2020" role="doc-biblioref">2020</a>)</span> and <span class="citation" data-cites="Arnekleiv2022">Arnekleiv et al. (<a href="#ref-Arnekleiv2022" role="doc-biblioref">2022</a>)</span>.</p>
<p>In this study we used data from years 2015 - 2020, and the total sample size across these years was 139 birds for the Aug-Jan period and 258 birds for the Feb-Jul period. We pooled data for males and females as survival was previously found to be very similar <span class="citation" data-cites="Israelsen2020">(<a href="#ref-Israelsen2020" role="doc-biblioref">Israelsen et al. 2020</a>)</span> and did not distinguish age classes for analysis.</p>
</section>
<section id="rodent-occupancy-data" class="level3">
<h3 class="anchored" data-anchor-id="rodent-occupancy-data">Rodent occupancy data</h3>
<p>As part of the line transect sampling (see above), observers are also requested to report whether they have seen any small rodents while surveying a transect. For each transect survey, this information is recorded as 1’s (small rodents spotted at least once) and 0’s (no small rodents spotted). We aggregated this data into area- and year-specific rodent occupancy covariates by averaging the 0 and 1 reports for all transect surveys within a given area and year and subsequently z-standardizing values. We note that while we refer to the covariate as “rodent occupancy” throughout the manuscript, it can be interpreted as an index for rodent abundance.</p>
</section>
</section>
<section id="national-scale-integrated-model" class="level2">
<h2 class="anchored" data-anchor-id="national-scale-integrated-model">National-scale integrated model</h2>
<section id="integrated-distance-sampling-model-idsm-for-willow-ptarmigan" class="level3">
<h3 class="anchored" data-anchor-id="integrated-distance-sampling-model-idsm-for-willow-ptarmigan">Integrated distance sampling model (IDSM) for willow ptarmigan</h3>
<p><span class="citation" data-cites="nilsen2023">E. B. Nilsen and Nater (<a href="#ref-nilsen2023" role="doc-biblioref">2024</a>)</span> recently developed an integrated distance sampling model (IDSM) which jointly analyses line transect and radio-telemetry data and applied it to willow ptarmigan in the Western part of Lierne municipality in Norway. The model consists of a population model with two age classes (juveniles and adults) and four data likelihoods: 1) likelihood for observation distances from transect lines for estimating detection probability; 2) likelihood for age-specific counts on transect surveys for estimating numbers of juveniles and adults present; 3) likelihood for juvenile to adult ratios observed at the locality level to provide estimate recruitment rate (as juveniles/adult); and 4) likelihood for known-fate telemetry data to estimate seasonal and annual survival. Below, we describe our new extension of this model to include data from several areas as opposed to just one. For more detailed information on the single-site model, including tests of model performance, see <span class="citation" data-cites="nilsen2023">E. B. Nilsen and Nater (<a href="#ref-nilsen2023" role="doc-biblioref">2024</a>)</span>.</p>
</section>
<section id="multi-area-model-extension" class="level3">
<h3 class="anchored" data-anchor-id="multi-area-model-extension">Multi-area model extension</h3>
<p>For applying the ptarmigan IDSM across all 41 reporting districts we included an area index in all model parameters (<a href="#fig-model" class="quarto-xref">Figure&nbsp;1</a>) and enabled sharing of information among areas by explicitly modelling spatial variation alongside shared temporal and residual variation in vital rates and detection parameters.</p>
<p>The spatially-explicit formulation of the two age-class population model can be written as:</p>
<p><span class="math display">\[
\begin{aligned}
  D_{juv, x, j, t+1} &amp; = D_{ad, x, j, t+1} * R_{x, t+1} \\
  D_{ad, x, j, t+1} &amp; = S_{x, t} * (D_{juv, x, j, t} + D_{ad, x, j, t})  
\end{aligned}
\]</span></p>
<p>Here, <span class="math inline">\(D_{juv, j, x, t}\)</span> and <span class="math inline">\(D_{ad, j, x, t}\)</span> are the densities of juvenile and adult ptarmigan in survey site (= transect) <span class="math inline">\(j\)</span> of area <span class="math inline">\(x\)</span> in year <span class="math inline">\(t\)</span>, respectively. Both juveniles and adult survive from year <span class="math inline">\(t\)</span> to <span class="math inline">\(t+1\)</span> with an area- (<span class="math inline">\(x\)</span>) and year- (<span class="math inline">\(t\)</span>) specific survival probability <span class="math inline">\(S_{x, t}\)</span>, and survivors produce the next generation of juveniles according to an area- and year-specific recruitment rate (<span class="math inline">\(R_{x,t}\)</span>).</p>
<div id="fig-model" class="quarto-figure quarto-figure-center quarto-float anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-model-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="Figures/RypeIDSM_multiArea_structure.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-model-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;1: Graphical representation of the ptarmigan life cycle with two age classes and the data sources, as well as joint distributions, included in the integrated distance sampling model. Indices <span class="math inline">\(t\)</span> denote year while <span class="math inline">\(x\)</span> represent the added dimension for area (corresponding to 41 reporting districts). Juvenile<sub>x,t</sub> = juveniles in area <span class="math inline">\(x\)</span> year <span class="math inline">\(t\)</span>. Adult<sub>x,t</sub> = adults in area <span class="math inline">\(x\)</span> in year <span class="math inline">\(t\)</span>. <span class="math inline">\(R_{x,t}\)</span> = recruitment rate in area <span class="math inline">\(x\)</span> in year <span class="math inline">\(t\)</span>. <span class="math inline">\(S_{x,t}\)</span> = survival probability from year <span class="math inline">\(t\)</span> to <span class="math inline">\(t+1\)</span> in area <span class="math inline">\(x\)</span>. Note that the additional site (=transect) dimension, <span class="math inline">\(j\)</span>, is omitted for the sake of illustration.
</figcaption>
</figure>
</div>
<p>The initial densities of adults, <span class="math inline">\(D_{ad, x, j, 1}\)</span>, are modelled for each site (= transect) as random realizations of log normal distributions with area-specific log means (<span class="math inline">\(\mu^{D1}_x\)</span>) and log standard deviations (<span class="math inline">\(\sigma^{D1}_x\)</span>). Site-specific initial juvenile density, <span class="math inline">\(D_{juv, x, j, 1}\)</span>, is calculated from initial adult density as <span class="math inline">\(D_{ad, x, j, 1} * R_{x, 1}\)</span>. Survival (<span class="math inline">\(S_{x,t}\)</span>) and recruitment (<span class="math inline">\(R_{x,t}\)</span>), on the other hand, are assumed to be the same for all sites <span class="math inline">\(j\)</span> within a given area <span class="math inline">\(x\)</span> and were modelled as:</p>
<p><span class="math display">\[
\begin{aligned}
  logit(S_{x, t}) &amp; = logit(\mu^S) + \varepsilon^{X.S}_{x} + \varepsilon^{T.S}_{t} + \varepsilon^{R.S}_{x,t} \\
    log(R_{x, t}) &amp; = log(\mu^R) + \beta_x*rodentOcc_{x,t} +  \varepsilon^{X.R}_{x} + \varepsilon^{T.R}_{t} + \varepsilon^{R.R}_{x,t}
\end{aligned}
\]</span> The global means ,<span class="math inline">\(\mu\)</span>, and the normally distributed spatial random effects, <span class="math inline">\(\varepsilon^X\)</span>, represent the equivalent of what is elsewhere referred to as “hyper-parameter distributions” for sharing information on demographic rates across areas <span class="citation" data-cites="horswill2019 horswill2021">(e.g. <a href="#ref-horswill2019" role="doc-biblioref">Horswill et al. 2019</a>, <a href="#ref-horswill2021" role="doc-biblioref">2021</a>)</span>. We also used this same approach for defining the area-specific effects (<span class="math inline">\(\beta_x\)</span>) of local yearly rodent occupancy (<span class="math inline">\(rodentOcc_{x,t}\)</span>) on recruitment. In addition to spatial variation in survival and recruitment, we also included large-scale temporal variation through random year effects that were shared by across all areas (<span class="math inline">\(\varepsilon^T_t\)</span>) and otherwise unaccounted for variation through year- and area-specific residual random effects (<span class="math inline">\(\varepsilon^R_{x,t}\)</span>). Spatial, temporal, and residual random effects were modelled as normally distributed with globally defined (= shared) standard deviations.</p>
<p>The three likelihoods for data resulting from the line transect sampling (observation distances, age-specific counts, and juvenile to adult ratios, see above) were also formulated as spatially explicit, with year- and area-specific distance sampling detection parameters modelled in the same way as survival and recruitment (except the effect of rodent occupancy, <a href="#fig-model" class="quarto-xref">Figure&nbsp;1</a>). For the known-fate telemetry data (and the seasonal decomposition of survival estimated from it), on the other hand, we did not add an additional area dimension as this data was only available for one study area (the Lierne area).</p>
</section>
<section id="model-implementation" class="level3">
<h3 class="anchored" data-anchor-id="model-implementation">Model implementation</h3>
<p>We implemented our multi-area IDSM in a Bayesian framework using NIMBLE version 1.2.0 <span class="citation" data-cites="valpine2017">(<a href="#ref-valpine2017" role="doc-biblioref">Valpine et al. 2017</a>)</span> in R version 4.4.0 <span class="citation" data-cites="R2024">(<a href="#ref-R2024" role="doc-biblioref">R Core Team 2024</a>)</span>. For the likelihood for line transect observation distances we used a custom half-normal distribution developed by Michael Scroggie in the “nimbleDistance” package (https://github.com/scrogster/nimbleDistance, see package vignette for specifics). We used non-informative uniform priors for all parameters, but used biologically sensible boundaries where possible. We simulated complete sets of initial values for all model nodes prior to model running and using pre-defined seeds to ensure reproducibility. Using NIMBLE’s standard samplers, we then ran 5 MCMC chains of 200k iterations each. We discarded the first 110k samples of each chain as burn-in, and thinned the remainder by a factor 30, resulting in a final joint posterior containing a total of 5 x 3k = 15k samples (note that high thinning rates were necessary to constrain memory load of the joint posterior, which included 5141 monitored parameters). In addition to the main model run, we implemented a second version of the model that did not use telemetry data to assess the potential impacts of auxiliary data from a single location on parameter estimates.</p>
</section>
</section>
<section id="follow-up-analyses" class="level2">
<h2 class="anchored" data-anchor-id="follow-up-analyses">Follow-up analyses</h2>
<section id="post-hoc-variance-decomposition" class="level3">
<h3 class="anchored" data-anchor-id="post-hoc-variance-decomposition">Post-hoc variance decomposition</h3>
<p>Following model fitting, we calculated posterior distributions for the proportions of variance in survival probabilities, recruitment rates, and detection decay explained by 1) spatial variation (<span class="math inline">\(var_{area}\)</span>), 2) temporal variation (<span class="math inline">\(var_{year}\)</span>), 3) residual variation (<span class="math inline">\(var_{residual}\)</span>), and 4) variation in rodent occupancy (<span class="math inline">\(var_{rodent}\)</span>). To obtain the proportion of variance explained by each of the component, we divided it by the sum of all the components (<span class="math inline">\(var_{area} + var_{year} + var_{residual} + var_{rodent}\)</span>). The spatial, temporal, and residual variance components were defined as the square of the estimated corresponding random effects standard deviation from the model while <span class="math inline">\(var_{rodent}\)</span> was calculated as the variance of all area- and year-specific <span class="math inline">\(\beta_x*rodentOcc_{x,t}\)</span> products. This approach for variance decomposition is equivalent to that used by <span class="citation" data-cites="nater2018">Chloé R. Nater et al. (<a href="#ref-nater2018" role="doc-biblioref">2018</a>)</span> and inspired by <span class="citation" data-cites="nakagawa2013">Nakagawa and Schielzeth (<a href="#ref-nakagawa2013" role="doc-biblioref">2013</a>)</span>.</p>
</section>
<section id="calculation-of-additional-demographic-parameters" class="level3">
<h3 class="anchored" data-anchor-id="calculation-of-additional-demographic-parameters">Calculation of additional demographic parameters</h3>
<p>For presentation and interpretation of results, we calculated additional key demographic parameters from the model posteriors. First, we calculated area- and year-specific average population densities, <span class="math inline">\(meanDens_{x, t}\)</span>, by averaging <span class="math inline">\(D_{juv, x, j, t} + D_{ad, x, j, t}\)</span> over all transect lines <span class="math inline">\(j\)</span> within area <span class="math inline">\(x\)</span> in year <span class="math inline">\(t\)</span>. We then proceeded to derive area- and year-specific realized population growth rates as <span class="math inline">\(meanDens_{x, t+1} / meanDens_{x, t}\)</span>. Additionally, we calculated generation time for each area using two different approaches: per-generation population growth rate <span class="citation" data-cites="caswell2000">(<a href="#ref-caswell2000" role="doc-biblioref">Caswell 2000</a>)</span> and elasticity of asymptotic growth rate with respect to fecundity <span class="citation" data-cites="brooks2001">(<a href="#ref-brooks2001" role="doc-biblioref">Brooks and Lebreton 2001</a>)</span>.</p>
</section>
<section id="parameter-and-sampling-correlations" class="level3">
<h3 class="anchored" data-anchor-id="parameter-and-sampling-correlations">Parameter and sampling correlations</h3>
<p>We also conducted correlation analyses on the posterior samples to a) check for potential evidence for vital rate trade-offs and/or density dependence and b) assess to what degree the former may be masked by sampling correlation. For a), we calculated Pearson’s correlation coefficients between area-specific time series of estimated vital rate, population density, and population growth rate for each posterior sample. For b), we calculated Pearson’s correlation coefficients between area- and time-dependent survival probabilities and recruitment rates across all posterior samples.</p>
</section>
</section>
<section id="reproducible-workflows-with-r-targets-and-nix" class="level2">
<h2 class="anchored" data-anchor-id="reproducible-workflows-with-r-targets-and-nix">Reproducible workflows with “R targets” and “Nix”</h2>
<p>Reproducibility and ease of repeating analyses was a key focus when developing the multi-area IDSM. To that end, we built a function-based workflow (<a href="#fig-workflow" class="quarto-xref">Figure&nbsp;2</a>) that includes a variety of options for controlling modelling decisions such as the year range of data to consider, the level of spatial aggregation (i.e.&nbsp;reporting district vs.&nbsp;survey locality), whether to model time variation in survival and/or effects of rodent occupancy, whether to run MCMC chains sequentially or in parallel, etc. We then set up three pipelines for executing the workflow that differ in their degree of automation and user interface to meet different needs and resource constraints. The first is a step-by-step R script to be run manually that is suitable for exploration, development, and debugging. The second is a largely automated “R targets” pipeline <span class="citation" data-cites="landau2021">(<a href="#ref-landau2021" role="doc-biblioref">Landau 2021</a>)</span>, which allows executions through a single command and maximizes efficiency by keeping track of the “up-to-date” vs.&nbsp;“outdated” status of different steps in the workflow. The third is a pipeline that is run directly from command line, sets up a fully reproducible environment through Nix [nix2004], and parallelizes the MCMC outside of R using GNUparallel <span class="citation" data-cites="tange2024">(<a href="#ref-tange2024" role="doc-biblioref">Tange 2024</a>)</span>. This option avoids a range of issues that can arise with R’s internal parallelization (e.g.&nbsp;processes running even when the parent R session has been restarted, hard to debug, bad error handling, etc.) and is particularly well suited for running on servers and high-performance computing clusters. For more details on pipeline implementations and options, we refer the reader to the vignettes in the GitHub repository: <a href="https://github.com/ErlendNilsen/OpenPop_Integrated_DistSamp" class="uri">https://github.com/ErlendNilsen/OpenPop_Integrated_DistSamp</a>.</p>
<div id="fig-workflow" class="quarto-figure quarto-figure-center quarto-float anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-workflow-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="Figures/Workflow_layout.png" class="img-fluid figure-img" width="424">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-workflow-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;2: Simplified graphical representation of the “targets pipeline” for the multi-area modelling setup. Upward facing triangles are functions, downward facing triangles are general options/arguments, circles are objects and outputs (=“targets”) created as part of the workflow. For the full representation, download the repository and call “targets::tar_visnetwork()”. The manual R and Nix/GNUparallel implementations of the workflow have the same structure and built on the same functions and relationships.
</figcaption>
</figure>
</div>
</section>
</section>
<section id="results" class="level1">
<h1>Results</h1>
<p>MCMC for all central model parameters converged within the given number of iterations, but chain mixing was remained suboptimal even at convergence for some parameters. The inclusion of telemetry data from a single location into the integrated model had did not substantially affect parameter estimates beyond the seasonal decomposition of survival in Lierne, where the telemetry data was collected (see online supplementary “Comparison_noTelemetry”). All numerical results in the following are presented as median [95% credible interval] unless otherwise indicated. Posterior summaries (median, 95% credible interval, mean, standard deviation, coefficient of variation) for all main parameters are also provided in the supplementary file “PosteriorSummaries_byAreas.csv”. Supplementary figures (SFs) are provided as .pdf files with captions in “SuppFigures_Captions.txt”; all files are deposited on OSF (<a href="https://osf.io/7326r/" class="uri">https://osf.io/7326r/</a>).</p>
<section id="population-density" class="level2">
<h2 class="anchored" data-anchor-id="population-density">Population density</h2>
<p>Only during the most recent four years (2018-2021) has data been collected regularly for all reporting areas included in the analyses. During this period, estimated population densities varied between 2.222 [1.564, 3.103] birds/km<sup>2</sup> in the area “Statskog og Klinga utm.” close to the coast in central Norway to 55.917 [51.814, 60.034] birds/km<sup>2</sup> in “Ålen og Haltdalen Fjellstyre” further south near the Swedish border. In general, recent population density appeared to be lowest in northern Norway and highest in the eastern part of central Norway (<a href="#fig-densA" class="quarto-xref">Figure&nbsp;3 (a)</a>). Uncertainty in density estimates was relatively consistent, with a few areas (including the one with the lowest estimated density, “Statskog og Klinga utm.”) sticking out by having substantially less precise estimates (<a href="#fig-densB" class="quarto-xref">Figure&nbsp;3 (b)</a>). Populations fluctuated substantially over time in any given area (SF “TimeSeries_popDens1.pdf”) and some years seemed to be indicative of relatively high (e.g.&nbsp;2011, 2014, 2018) or low (e.g.&nbsp;2012, 2015) densities across a substantial number of areas.</p>
<div id="fig-dens" class="quarto-layout-panel">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-dens-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="quarto-layout-row">
<div class="quarto-layout-cell-subref quarto-layout-cell" data-ref-parent="fig-dens" style="flex-basis: 50.0%;justify-content: flex-start;">
<div id="fig-densA" class="quarto-figure quarto-figure-center quarto-float anchored">
<figure class="quarto-float quarto-subfloat-fig figure">
<div aria-describedby="fig-densA-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="Figures/Avg_popDens_Map-4.png" class="img-fluid figure-img" data-ref-parent="fig-dens">
</div>
<figcaption class="quarto-float-caption-bottom quarto-subfloat-caption quarto-subfloat-fig" id="fig-densA-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
(a) Median density estimates
</figcaption>
</figure>
</div>
</div>
<div class="quarto-layout-cell-subref quarto-layout-cell" data-ref-parent="fig-dens" style="flex-basis: 50.0%;justify-content: flex-start;">
<div id="fig-densB" class="quarto-figure quarto-figure-center quarto-float anchored">
<figure class="quarto-float quarto-subfloat-fig figure">
<div aria-describedby="fig-densB-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="Figures/Avg_popDens_Map-6.png" class="img-fluid figure-img" data-ref-parent="fig-dens">
</div>
<figcaption class="quarto-float-caption-bottom quarto-subfloat-caption quarto-subfloat-fig" id="fig-densB-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
(b) Uncertainty in density estimates
</figcaption>
</figure>
</div>
</div>
</div>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-dens-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;3: Median (a) and coefficient of variation (standard deviation / mean) (b) of posterior estimates of average ptarmigan density in the four most recent years (2018-2021) across 41 reporting areas (summarised at the municipality level) in Norway. Darker colors indicate higher median values and higher uncertainty.
</figcaption>
</figure>
</div>
</section>
<section id="population-growth-rate" class="level2">
<h2 class="anchored" data-anchor-id="population-growth-rate">Population growth rate</h2>
<p>Average population growth rates over the last four years (2018-2021) ranged from moderate declines (0.716 [0.639, 0.934] in the “Kongsvoll” area) to &gt; 50% increase (1.551 [1.271, 1.953] in the “Statskog og Klinga utm.” area). In the majority of reporting areas (23 out of 41), populations of willow ptarmigan have been increasing over the period 2018-2021 (<a href="#fig-lam" class="quarto-xref">Figure&nbsp;4</a>). Some areas – predominantly in central Norway – also had declining populations, but many of those declines followed upon periods of increase between the start of data collection in 2007 and sometime between 2016 and 2018 (SF “TimeSeries_popDens1.pdf”).</p>
<div id="fig-lam" class="quarto-layout-panel">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-lam-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="quarto-layout-row">
<div class="quarto-layout-cell-subref quarto-layout-cell" data-ref-parent="fig-lam" style="flex-basis: 50.0%;justify-content: flex-start;">
<div id="fig-lamA" class="quarto-figure quarto-figure-center quarto-float anchored">
<figure class="quarto-float quarto-subfloat-fig figure">
<div aria-describedby="fig-lamA-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="Figures/Avg_lambda_Map-4.png" class="img-fluid figure-img" data-ref-parent="fig-lam">
</div>
<figcaption class="quarto-float-caption-bottom quarto-subfloat-caption quarto-subfloat-fig" id="fig-lamA-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
(a) Median population growth rate estimates
</figcaption>
</figure>
</div>
</div>
<div class="quarto-layout-cell-subref quarto-layout-cell" data-ref-parent="fig-lam" style="flex-basis: 50.0%;justify-content: flex-start;">
<div id="fig-lamB" class="quarto-figure quarto-figure-center quarto-float anchored">
<figure class="quarto-float quarto-subfloat-fig figure">
<div aria-describedby="fig-lamB-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="Figures/Avg_lambda_Map-5.png" class="img-fluid figure-img" data-ref-parent="fig-lam">
</div>
<figcaption class="quarto-float-caption-bottom quarto-subfloat-caption quarto-subfloat-fig" id="fig-lamB-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
(b) Uncertainty in population growth rate estimates
</figcaption>
</figure>
</div>
</div>
</div>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-lam-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;4: Median (a) and standard deviation (b) of posterior estimates of average annual population growth rate over the four most recent years (2018-2021) across 41 reporting areas (summarised at the municipality level) in Norway. In a), pinkish colors indicate declining populations while greenish colors indicate growing populations (white = stable populations). In b) darker colors indicate higher uncertainty.
</figcaption>
</figure>
</div>
<p>The highest recent population growth rates were estimated for areas with relatively low recent population densities across latitudes but we did not find evidence for a strong association between population growth rates and population densities across areas in general (<a href="#fig-paramCorrs" class="quarto-xref">Figure&nbsp;5</a> A). Within areas, however, we found substantial negative relationships between population density and population growth rates, with median correlation coefficients ranging from -0.219 to -0.949 (supplementary file “DD_corrCoef.csv”).</p>
<div id="fig-paramCorrs" class="quarto-figure quarto-figure-center quarto-float anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-paramCorrs-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="Figures/AllParams_Latitude.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-paramCorrs-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;5: Posterior summaries (points = posterior medians, lines = 95% credible intervals) of area-specific population growth rate vs.&nbsp;population density over the four most recent years (2018-2021, A), recruitment rate vs.&nbsp;survival probability (B) and rodent effect on recruitment along a longitudinal gradient (C). Color indicates latitude of the midpoint of each area.
</figcaption>
</figure>
</div>
</section>
<section id="survival-probabilities-recruitment-rates-and-generation-times" class="level2">
<h2 class="anchored" data-anchor-id="survival-probabilities-recruitment-rates-and-generation-times">Survival probabilities, recruitment rates, and generation times</h2>
<p>Annual survival probabilities ranged from 0.329 [0.254, 0.396] (area “Statskog og Klinga utm.”) to 0.428 [0.365, 0.503] (area “Øst Finnmark”) across reporting areas in Norway, with the highest values occurring in the far north and in the mountains in the south (<a href="#fig-pVRA" class="quarto-xref">Figure&nbsp;6 (a)</a>). The global average survival probability across all areas and years (<span class="math inline">\(\mu^S\)</span>) was estimated at 0.379 [0.362, 0.397]. Spatial variation in survival (random effect SD on logit scale = 0.158 [0.064, 0.247]) was relatively low compared to temporal (0.586 [0.397, 0.932]) and residual (0.638 [0.579, 0.705]) variation.</p>
<p>Recruitment rates varied between 1.721 [1.439, 1.98] (area “Gausdal Fjellstyre”) and 3.203 [2.602, 3.951] (area “Statskog og Klinga utm.”) and displayed a spatial pattern opposite to that of annual survival, i.e., lower recruitment rates co-occurring with higher survival rates and vice-versa (<a href="#fig-pVRB" class="quarto-xref">Figure&nbsp;6 (b)</a>; <a href="#fig-paramCorrs" class="quarto-xref">Figure&nbsp;5</a> B). Across all areas and years, average recruitment rate was 2.396 [2.226, 2.651]. Unlike for survival, the model predicted similar magnitudes of spatial and temporal variation (random effect SDs on log scale of 0.16 [0.121, 0.216] and 0.117 [0.072, 0.193], respectively), and about twice as much residual variation (0.332 [0.308, 0.358]).</p>
<div id="fig-pVR" class="quarto-layout-panel">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-pVR-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="quarto-layout-row">
<div class="quarto-layout-cell-subref quarto-layout-cell" data-ref-parent="fig-pVR" style="flex-basis: 50.0%;justify-content: flex-start;">
<div id="fig-pVRA" class="quarto-figure quarto-figure-center quarto-float anchored">
<figure class="quarto-float quarto-subfloat-fig figure">
<div aria-describedby="fig-pVRA-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="Figures/Avg_pSurv_Map-1.png" class="img-fluid figure-img" data-ref-parent="fig-pVR">
</div>
<figcaption class="quarto-float-caption-bottom quarto-subfloat-caption quarto-subfloat-fig" id="fig-pVRA-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
(a) Median survival probabilities
</figcaption>
</figure>
</div>
</div>
<div class="quarto-layout-cell-subref quarto-layout-cell" data-ref-parent="fig-pVR" style="flex-basis: 50.0%;justify-content: flex-start;">
<div id="fig-pVRB" class="quarto-figure quarto-figure-center quarto-float anchored">
<figure class="quarto-float quarto-subfloat-fig figure">
<div aria-describedby="fig-pVRB-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="Figures/Avg_rRep_Map-1.png" class="img-fluid figure-img" data-ref-parent="fig-pVR">
</div>
<figcaption class="quarto-float-caption-bottom quarto-subfloat-caption quarto-subfloat-fig" id="fig-pVRB-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
(b) Median recruitment rates
</figcaption>
</figure>
</div>
</div>
</div>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-pVR-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;6: Posterior medians of average annual survival probabilities (a) and recruitment rates (b) across 41 reporting areas (summarised at the municipality level) in Norway. Darker colors indicate higher median values. Measures for corresponding uncertainty in estimates are visualized in SFs “Avg_pSurv_Map.pdf” and “Avg_rRep_Map.pdf” for survival and recruitment, respectively.
</figcaption>
</figure>
</div>
<p>The MCMC chains for many of the area-specific average survival probabilities and recruitment rates, as well as for the global averages for both vital rates, were mixing rather poorly. Despite that, mixing was good and resulting posteriors well defined for the area- and year-specific estimates of survival and recruitment (SF “PostDens_tS_tR.pdf”). There was substantial variation in both vital rates across time (SFs “TimeSeries_pSurv.pdf” and “TimeSeries_rRep.pdf”). In a substantial number of areas, the years 2011, 2014, and 2018 not only supported high population densities (see above) but were also characterized by both high recruitment and low subsequent survival. The overall low density years 2012 and 2015, conversely, often featured lower recruitment and, in some cases, higher survival. Notably, there were also years with very little spatial synchrony, i.e.&nbsp;very different relative yearly survival probabilities and recruitment rates (e.g.&nbsp;2010 and 2020 for survival and 2013, 2016, and 2017 for recruitment). This same pattern was also reflected in the within-sample correlations between population density in vital rates, which were predominantly positive for recruitment and negative for subsequent survival (supplementary file “DD_corrCoef.csv”). Sampling correlation between annual recruitment rates and survival probabilities was moderate when no time lag was considered (<span class="math inline">\(R_t\)</span> vs.&nbsp;<span class="math inline">\(S_t\)</span>, average coefficient = -0.399) and very low when comparing survival to subsequent recruitment (<span class="math inline">\(R_t\)</span> vs.&nbsp;<span class="math inline">\(S_t\)</span>, average coefficient = NA). Correlation coefficients varied substantially across areas though, featuring both positive and negative values with no clear spatial pattern (SF “SurvRepCorr_Latitude.pdf”, supplementary file “VR_corrCoef.csv”).</p>
<p>Based on estimates of population growth rates and vital rates, we also calculated generation time as both per-generation population growth rate (<span class="math inline">\(R0\)</span>) and inverse of fecundity elasticity (<span class="math inline">\(1/elas_F\)</span>). The two approaches yielded very similar estimates (median correlation coefficient = 0.953) ranging from 1.312, [1.253, 1.384] to 1.639, [1.557, 1.74] years across areas (SF “GenerationTime_Latitude.pdf”). Spatial patterns in generation time were consistent with those for survival and recruitment, with the highest values occurring in the North and in the mountaineous regions in central Norway (SFs “GenerationTime_R0_Map.pdf” and “GenerationTime_elasF_Map.pdf”).</p>
</section>
<section id="effects-of-rodent-occupancy" class="level2">
<h2 class="anchored" data-anchor-id="effects-of-rodent-occupancy">Effects of rodent occupancy</h2>
<p>The model predicted a positive global effect of rodent occupancy on recruitment rate (average slope on the log scale = 0.067 [0.01, 0.125]). Nonetheless, spatial variation in the rodent effect was non-negligible (random effect SD = 0.084 [0.004, 0.148]). This resulted in negative (median) effects in 3 areas, positive (median) effects in 38 areas, and a range of effect sizes from -0.018 [-0.187, 0.104] (area “Selbu Fjellstyre”) to 0.144 [0.023, 0.315] (area “Øst Finnmark”, <a href="#fig-paramCorrs" class="quarto-xref">Figure&nbsp;5</a> C, SF “Rep_betaR.R.pdf”). The largest positive rodent effects were estimated for areas in the very North of Norway, as well as in the mountaineous regions in the central and southwestern parts of the country (SF “betaR_Map.pdf”). Effects with negative posterior medians were located mostly at intermediate latitudes, but we note that all of these had posterior distributions featuring substantial overlap with 0 (<a href="#fig-paramCorrs" class="quarto-xref">Figure&nbsp;5</a> C).</p>
</section>
<section id="detection-parameters" class="level2">
<h2 class="anchored" data-anchor-id="detection-parameters">Detection parameters</h2>
<p>Detection decay parameters, which determine detection probability in distance sampling surveys, varied across areas from between 68.074 [61.154, 75.776] in “Namskogan Fjellstyre” to 119.974 [108.513, 133.262] in “Engerdal Fjellstyre, resulting in detection probabilities over the transect sites ranging from 0.427 [0.383, 0.475] to 0.752 [0.68, 0.835], respectively (truncation distance = 200 m) . The global average detection decay was 92.187 [86.588, 98.071] (detection probability = 0.578 [0.543, 0.615]), and in general, higher values were more common in the Southern half of the country than the Northern half (SF”Avg_detect_Map.pdf”). Variation in detection over time was modest on average but the degree of temporal changes varied by area, with some areas having nearly constant detection while others showed variation by factors larger than 1.5 (SF “TimeSeries_pDetect.pdf”). The estimated average among-year variation in detection decay (random effect log SD = 0.074 [0.047, 0.123]) was lower than spatial (0.142 [0.109, 0.186]) and residual (0.136 [0.119, 0.154]) variation.</p>
</section>
<section id="variance-decomposition" class="level2">
<h2 class="anchored" data-anchor-id="variance-decomposition">Variance decomposition</h2>
<p>The relative importance of different components for explaining parameter variation differed among recruitment rate, survival probability, and detection decay (<a href="#fig-varDecomp" class="quarto-xref">Figure&nbsp;7</a>). The largest portion of variation in recruitment was attributed to residual variation (67.5 [55.7, 78] %), followed by spatial (15.7 [9.3, 25.7] %) and temporal (8.4 [3.2, 20.5] %) variation. Rodent occupancy, which contains both a spatial and a temporal dimension, explained 7 [1.4, 13.7] % of the total variation. For survival, there was large uncertainty in the estimated proportions of variance explained by different components. The model predicted similar potential contributions from temporal (44.2 [26, 67.4] %) and residual (52.4 [30.7, 70] %) variation and suggested that spatial variation was only responsible for 3.1 [0.5, 8.2] % of the total variance. The majority of variance in detection decay was attributed evenly to spatial and residual variation at 45.1 [30.8, 60.7] % and 41 [28.4, 54.6] %, respectively. Temporal factors only accounted for 12.5 [5.1, 28.9] %, of detection variation.</p>
<div id="fig-varDecomp" class="quarto-figure quarto-figure-center quarto-float anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-varDecomp-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="Figures/VarDecomposition_overall.png" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-varDecomp-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;7: Posterior distributions for the proportions of parameter variance explained by spatial (blue), temporal (orange), and residual (green) variation, as well as by effects of rodent occupancy (yellow).
</figcaption>
</figure>
</div>
</section>
</section>
<section id="discussion" class="level1">
<h1>Discussion</h1>
<p>Building on the work of <span class="citation" data-cites="nilsen2023">E. B. Nilsen and Nater (<a href="#ref-nilsen2023" role="doc-biblioref">2024</a>)</span>, we applied a novel integrated population model to data collected through a national-scale participatory monitoring programme to estimate spatial and temporal variation in demography of a culturally important game bird species, the willow ptarmigan. While our study was exploratory in nature, it recovered patterns consistent with ecological and life-history theory including trade-offs between survival and recruitment, and a tendency towards slower life histories at higher latitudes and altitudes. Space-for-time substitution also provided the statistical power necessary for the analysis to provide evidence for the alternative prey hypothesis, i.e.&nbsp;ptarmigans benefiting from high abundance of alternative rodent prey for their predators. Taken together, the results highlight the potential of integrating demographic data across large spatial scales in the contexts of both informing management and creating biodiversity indicators for higher-level reporting.</p>
<section id="abundance-and-vital-rates-across-space-and-time" class="level2">
<h2 class="anchored" data-anchor-id="abundance-and-vital-rates-across-space-and-time">Abundance and vital rates across space and time</h2>
<p>The wide spatial distribution of the line transect monitoring afforded us the opportunity to explore variation in population density and vital rates across a relatively large spatial extent.</p>
<p>Ptarmigan densities across the 41 reporting districts included in our analyses varied from around 2 birds/km<sup>2</sup> to 55 birds/km<sup>2</sup>, with the lowest densities occurring far north in the country, as well as on the west coast and in the mountains in central Norway <a href="#fig-densA" class="quarto-xref">Figure&nbsp;3 (a)</a>. The same spatial pattern was also evident at the level of the demographic rates: consistent with basic life history theory <span class="citation" data-cites="stearns1992">(<a href="#ref-stearns1992" role="doc-biblioref">Stearns 1992</a>)</span>, average recruitment rates were inversely related to average survival probabilities <a href="#fig-paramCorrs" class="quarto-xref">Figure&nbsp;5</a>, and the slower life histories (higher survival, lower recruitment, and longer generation times) were more common in the northern and mountaineous parts of the country. This aligns with previous studies reporting relatively slower bird life histories in alpine / high altitude areas <span class="citation" data-cites="sandercock2005 bears2009 Wilson_Martin_2011 alice2016">(e.g. <a href="#ref-sandercock2005" role="doc-biblioref">Sandercock, Martin, and Hannon 2005</a>; <a href="#ref-bears2009" role="doc-biblioref">Bears, Martin, and White 2009</a>; <a href="#ref-Wilson_Martin_2011" role="doc-biblioref">Wilson and Martin 2011</a>; <a href="#ref-alice2016" role="doc-biblioref">Alice Boyle, Sandercock, and Martin 2016</a>)</span>. In Norway, the northern and mountainous areas are characterized by more extreme climatic conditions, boasting cold temperatures and short growing seasons. Resulting reduced primary production limits food availability and as ptarmigan are income breeders that use food resources acquired from nesting areas to supply energy and nutrients for egg production and incubation <span class="citation" data-cites="sandercock2005">(<a href="#ref-sandercock2005" role="doc-biblioref">Sandercock, Martin, and Hannon 2005</a>)</span>, lower carrying capacity in such areas is to be expected.</p>
<p>We found increasing population trends over recent years in over half of the reporting districts, but population declines were also evident in some areas, particularly in the mountains in central Norway <a href="#fig-lamA" class="quarto-xref">Figure&nbsp;4 (a)</a>. Predominantly increasing population trends are consistent with a recent national-scale analysis by <span class="citation" data-cites="nilsen2020">E. B. Nilsen and Rød-Eriksen (<a href="#ref-nilsen2020" role="doc-biblioref">2020</a>)</span> which found an overall increase in the Norwegian ptarmigan population between 2009 and 2020. While we may speculate that recent population trends could be linked to changes in harvest regulations and/or climatic conditions, considering the whole time-series (2007-2021) illustrated that population densities in all areas were subject to substantial variation across years, featuring periods of stability, increase, and decrease (SF “TimeSeries_PopDens1.pdf”). In most areas, there were also strong year-by-year fluctuations in population density on top of longer-term trends. Some of the resulting “high density years” were highly synchronized across large spatial scales, such as the years 2011, 2013, and 2018. Taking a closer look, we find that these are years that are characterized by high recruitment (SF “TimeSeries_rRep.pdf”), followed by a low survival the year after (SF “TimeSeries_pSurv.pdf”). This often resulted in steep population declines towards the following year. The fact that these same years also match up with observed peaks in rodent abundance in many areas, together with the largely positive effects of rodent occupancy on recruitment estimated by our model (<a href="#fig-paramCorrs" class="quarto-xref">Figure&nbsp;5</a> C), provides evidence for the Alternative Prey Hypothesis [APH; <span class="citation" data-cites="hagen1952">Hagen (<a href="#ref-hagen1952" role="doc-biblioref">1952</a>)</span>]. The APH stipulates that high abundance of alternative prey (rodents, in this case) for common predators leads to population growth, and is well-supported throughout the literature for a range of taxa <span class="citation" data-cites="hagen1952 Kjellander_Nordström_2003 Reif_Tornberg_Jungell_Korpimäki_2001">(e.g., <a href="#ref-hagen1952" role="doc-biblioref">Hagen 1952</a>; <a href="#ref-Kjellander_Nordström_2003" role="doc-biblioref">Kjellander and Nordström 2003</a>; <a href="#ref-Reif_Tornberg_Jungell_Korpimäki_2001" role="doc-biblioref">Reif et al. 2001</a>)</span>, including willow ptarmigan <span class="citation" data-cites="bowler2020">(<a href="#ref-bowler2020" role="doc-biblioref">Bowler et al. 2020</a>)</span>. While <span class="citation" data-cites="Nyström_etal_2006">Nyström et al. (<a href="#ref-Nyström_etal_2006" role="doc-biblioref">2006</a>)</span> suggested that gyrfalcons, which are specialized ptarmigan predators, do not respond to rodent populations or switch to alternative prey when ptarmigan populations are low, generalist predators, such as red foxes, are likely to shift from preying on ptarmigans to rodents when the latter become abundant <span class="citation" data-cites="Breisjøberget_etal_2018 bowler2020">(e.g. <a href="#ref-Breisjøberget_etal_2018" role="doc-biblioref">Breisjøberget, Odden, Wegge, et al. 2018</a>; <a href="#ref-bowler2020" role="doc-biblioref">Bowler et al. 2020</a>)</span>. Taking a spatial perspective, the highest latitude and highest altitude areas stood out once more, sporting the strongest effects of rodent occupancy (SF “betaR_Map.pdf”). This could be related to warmer areas generally having larger predator guilds, and consequently more generalists that are able to maintain relatively stable populations irrespective of small rodent abundance <span class="citation" data-cites="bowler2020">(<a href="#ref-bowler2020" role="doc-biblioref">Bowler et al. 2020</a>)</span>.</p>
<p>Notably, the conclusive estimation of overall positive effects of rodent occupancy on recruitment in our model was only possible thanks to the integration and sharing of data across multiple areas. When <span class="citation" data-cites="nilsen2023">E. B. Nilsen and Nater (<a href="#ref-nilsen2023" role="doc-biblioref">2024</a>)</span> fit the IDSM to data from only a single area, they were unable to obtain a reliable estimate for the rodent effect due to limited statistical power. Consequently, the space-for-time substitution that comes with extending the model across multiple area allows estimation of covariate effects that otherwise cannot be estimated, and opens up for future possibilities for studying effects of not just rodents, but also other environmental drivers on ptarmigan population dynamics. Doing so may also help with better understanding the mechanisms underlying the large portion of demographic rate variation that could only be attributed to random variation so far. This is the case especially for the relatively large residual variation (<a href="#fig-varDecomp" class="quarto-xref">Figure&nbsp;7</a>) but also relevant for constant spatial and shared temporal variation. In previous work based on both marked <span class="citation" data-cites="Eriksen2023">(<a href="#ref-Eriksen2023" role="doc-biblioref">Eriksen et al. 2023</a>)</span> and unmarked birds <span class="citation" data-cites="bowler2020 Henden2020 Novoa2016">(<a href="#ref-bowler2020" role="doc-biblioref">Bowler et al. 2020</a>; <a href="#ref-Henden2020" role="doc-biblioref">J. A. Henden et al. 2020</a>; <a href="#ref-Novoa2016" role="doc-biblioref">Novoa et al. 2016</a>)</span>, spring conditions have come out as an important predictor of ptarmigan recruitment rates. In general, warmer and earlier springs seem to favour earlier breeding, larger clutch sizes <span class="citation" data-cites="Eriksen2023">(<a href="#ref-Eriksen2023" role="doc-biblioref">Eriksen et al. 2023</a>)</span>, and resulting higher recruitment rates measured in the late summer and early fall. <span class="citation" data-cites="bowler2020">Bowler et al. (<a href="#ref-bowler2020" role="doc-biblioref">2020</a>)</span> further reported that the strength of this relationship was not consistent in time and space, but was generally stronger in colder areas, similar to what we found for the effect of rodent occupancy here. In practice, measures representing spring conditions, such as the cover of ericaceous shrubs (a proxy for food availability) or spatially-explicit spring green up dates derived from remote-sensing data, thus constitute relevant candidate covariates for future work alongside temperature.</p>
<p>Another important determinant of vital rate variation is density dependence, in particular for exploited species like willow ptarmigan <span class="citation" data-cites="andrewartha1954 sandercock_etal2011 aanes2002 willebrand2001">(<a href="#ref-andrewartha1954" role="doc-biblioref">Andrewartha and Birch 1954</a>; <a href="#ref-sandercock_etal2011" role="doc-biblioref">Sandercock et al. 2011</a>; <a href="#ref-aanes2002" role="doc-biblioref">Aanes et al. 2002</a>; <a href="#ref-willebrand2001" role="doc-biblioref">Willebrand and Hörnell 2001</a>)</span>. Negative density dependence has been found in several gallinaceous birds such as northern bobwhites <em>Colinus virginianus</em> <span class="citation" data-cites="mcconnell2018">(<a href="#ref-mcconnell2018" role="doc-biblioref">McConnell et al. 2018</a>)</span>, <em>Perdix perdix</em> <span class="citation" data-cites="bro2003">(<a href="#ref-bro2003" role="doc-biblioref">Bro et al. 2003</a>)</span>, and wild turkeys <em>Meleagris gallopavo</em> <span class="citation" data-cites="mcghee2007">(<a href="#ref-mcghee2007" role="doc-biblioref">McGhee and Berkson 2007</a>)</span>). For willow ptarmigan, evidence for density-dependent population regulation has been mixed. <span class="citation" data-cites="myrberget1988">Myrberget (<a href="#ref-myrberget1988" role="doc-biblioref">1988</a>)</span>, for example, observed no change in productivity despite a 50% decrease in abundance, while <span class="citation" data-cites="pedersen2004">Pedersen et al. (<a href="#ref-pedersen2004" role="doc-biblioref">2004</a>)</span> reported strong negative density-dependence over winter and posited that dispersal may be the vital rate that responded to changes in density most strongly. Similarly, <span class="citation" data-cites="Henden2020">J. A. Henden et al. (<a href="#ref-Henden2020" role="doc-biblioref">2020</a>)</span> reported negative density dependence when using a Gompertz-model to examine how density and a range of environmental covariates affected willow ptarmigan population dynamics in the northernmost parts of Norway. While we did not explicitly model density dependence in this study, our results can provide some preliminary insights into potential density feedbacks from both a spatial (cross-population) and a temporal (within-population) angle. Comparing average population densities and growth rate across areas did not provide evidence for strong density dependence, but there was a tendency towards the highest population growth rates appearing in areas with relatively low density, and relatively low growth rates in high-density areas <a href="#fig-paramCorrs" class="quarto-xref">Figure&nbsp;5</a>. When considering density dependence across years within select areas, however, we found that higher density years were associated with higher recruitment the same year, but followed by lower apparent survival probabilities and, consequently, lower population growth rates (as determined by post-hoc Pearson correlation coefficients, supplementary file “DD_corrCoef.csv”). While this seems to support the notion of negative density-dependence, testing for this post-hoc gives results that are confounded with sampling correlation <span class="citation" data-cites="Freckleton2006">(<a href="#ref-Freckleton2006" role="doc-biblioref">Freckleton et al. 2006</a>)</span>. Our tests showed a moderate degree of sampling correlation between survival and recruitment on average (up to -0.399), but there was substantial variation in the degree of correlation across areas (SF “SurvRepCorr_Latitude.pdf”). Hence, formally modelling density-dependence, possibly using different forms and time-lags, could prove to be a promising extension of our modelling framework in the future.</p>
</section>
<section id="implications-for-management-and-monitoring" class="level2">
<h2 class="anchored" data-anchor-id="implications-for-management-and-monitoring">Implications for management and monitoring</h2>
<p>Management decisions made at the resolution of large geopolitical boundaries (e.g., Norway) run a high risk of being inadequate when there is substantial spatial variation in demographic processes and population dynamics, as is the case for willow ptarmigan. In Norway, willow ptarmigan – and small game in general – are managed at the local and/or regional scale, with rather limited national regulation beyond updating the length of the hunting season every fourth year. In effect, management system, regulation type (quota type, season length, number of licences, bag limit etc.), and quota size are governed by the local or regional stakeholders <span class="citation" data-cites="Eriksen2018 breisjøberget2018">(<a href="#ref-Eriksen2018" role="doc-biblioref">Eriksen, Moa, and Nilsen 2018</a>; <a href="#ref-breisjøberget2018" role="doc-biblioref">Breisjøberget, Odden, Storaas, et al. 2018</a>)</span>. Thus, while national estimates (abundance and/or temporal trend in abundance) might be important for red listing decisions and for setting the maximum hunting season length, remaining decisions about harvest management are taken locally. The results from our study highlight a large degree of spatio-temporal variation in both ptarmigan densities and demographic rates, suggesting that it is indeed suitable for management decisions to be spatially refined and ideally informed by up-to-date knowledge about recent “local” population processes. Accessible and easily repeatable modelling workflows, such as the one we have developed in this study, can thus become a valuable source of information for local decision-makers.</p>
<p>Our results also provided some insights into the value, and possibly opportunities for improving the monitoring programme. First and foremost, our study demonstrates the tremendous potential lying in coordinating structured monitoring that employs common sampling protocol, training programmes, and data processing pipelines. These were indeed the prerequisites that allowed us to easily and efficiently integrate data collected across the entire country in a joint analysis, and draw inference on fine-scale spatio-temporal variation in demography and population dynamics at across a large area. While overall less variable across space and time than vital rates, differences in detection probabilities were nonetheless evident (SFs “Avg_detect_Map.pdf” and “TimeSeries_pDetect.pdf”) and may help with mapping out potential for improvement in the monitoring programme. Particularly, we found generally lower detection probabilities in the northern half of Norway. This may be related to habitat features, as the transects in the North might be to a larger extent located in birch forests and rugged terrain, which may hamper detectability. Additionally, the slower life histories in the northern areas are reflected as generally smaller bird clusters as well, and smaller clusters have previously been shown to have a lower detectability than larger ones <span class="citation" data-cites="bowler2020">(e.g. <a href="#ref-bowler2020" role="doc-biblioref">Bowler et al. 2020</a>, see also next section)</span>. Our modelling framework can be easily adapted for studying the impact of these and other variables on detectability (see below). Additionally, our results could motivate taking a closer look at monitoring challenges and potential improvements in the northern part of the country in particular. Ultimately, increased detection probability would contribute to obtaining more precise estimates of both population density and demographic rates, which – in turn – would be of great value in particular in areas with relatively low population densities, low number of transects, and less years of data.</p>
</section>
<section id="model-limitations-and-outlook" class="level2">
<h2 class="anchored" data-anchor-id="model-limitations-and-outlook">Model limitations and outlook</h2>
<p>The primary focus of this work was placed on developing an effective pipeline for integrating data and modelling population dynamics across a large number of areas. Consequently, many additional opportunities for improving and refining the modelling framework itself remain. First, the precision and accuracy of model estimates might be increased through better accounting for heterogeneity and potential biases in detection of birds during the line transect surveys. In an earlier study analyzing data from the same monitoring programme, <span class="citation" data-cites="bowler2020">Bowler et al. (<a href="#ref-bowler2020" role="doc-biblioref">2020</a>)</span>, found that detection probability was not independent of the size of group birds were part of, resulting in birds in larger groups being more likely to be detected, especially at larger distances. When birds are observed in larger groups, it is also not unlikely that human observers may miscount, i.e.&nbsp;that there is some observation error in the number reported. This could be incorporated by including an additional layer of hierarchy to the observation process <span class="citation" data-cites="hamilton2018">(see e.g., <a href="#ref-hamilton2018" role="doc-biblioref">Hamilton et al. 2018</a>)</span>, and possibly further extended to also account for error in judging the observation distance <span class="citation" data-cites="marques2004">(e.g., <a href="#ref-marques2004" role="doc-biblioref">Marques 2004</a>)</span>. Another potential source of bias in our IDSM is related to failure to correctly assign the age class of observed birds. <span class="citation" data-cites="nilsen2023">E. B. Nilsen and Nater (<a href="#ref-nilsen2023" role="doc-biblioref">2024</a>)</span> showed that incorrect age assessment can bias (relative) estimates of survival and recruitment, and while they only found a weak bias in their case study on a single area, the problem may be larger in a multi-area setting that may contain areas with different proportions of misclassified observations. If misclassification happened at random, mixture models could be used to determine the likely age class of individuals to whom no age class was assigned during observations <span class="citation" data-cites="mccrea2013">(<a href="#ref-mccrea2013" role="doc-biblioref">McCrea, Morgan, and Cole 2013</a>)</span>. In our case, we might suspect that an observer is more likely to classify an adult bird as juvenile rather than the other way around, and more likely to assign “unknown” age class to juveniles than adults. One reason for this is that observers look for specific signs to classifying a bird as adult (e.g.&nbsp;size, male sound), and might default to juvenile or unknown if the signs are not clearly detected. Future studies should investigate to what degree available information on e.g.&nbsp;group composition could be used for this, and what kind of auxiliary data would need to be collected to reliably model misclassification error.</p>
<p>The second (and perhaps most attractive) aspect of our modelling framework in the context of future work is its spatio-temporal hierarchical structure. While we included spatial, temporal, and residual variation in our framework here, we treated them as independent. Alternatively, spatial (and temporal) correlations among parameters can be modelled explicitly, something that is commonly done e.g.&nbsp;in modern species distribution models <span class="citation" data-cites="pacifici2017 guelat2018">(e.g. <a href="#ref-pacifici2017" role="doc-biblioref">Pacifici et al. 2017</a>; <a href="#ref-guelat2018" role="doc-biblioref">Guélat and Kéry 2018</a>)</span>. For demographic models, this has rarely been implemented so far, not least due to the fact that few demographic models have sufficient spatial resolution <span class="citation" data-cites="schaub2021">(<a href="#ref-schaub2021" role="doc-biblioref">Schaub and Kéry 2021</a>)</span>. The ptarmigan IDSM presented in this study, however, does have sufficient resolution and our results do indeed support that there is spatial clustering in both overall and time-dependent demographic parameters (e.g. <a href="#fig-pVR" class="quarto-xref">Figure&nbsp;6</a>, SFs “Avg_pSurv_Map.pdf” &amp; “Avg_rRep_Map.pdf”). Furthermore, we did find that mixing of several of the global and area-specific intercept parameters in the current model was suboptimal, suggesting that there may be much to gain from additional structuring, as well as from development of more efficient MCMC sampling strategies for the resulting extended model. One promising framework for approaching this are conditionally autoregressive models <span class="citation" data-cites="verhoef2018">(CARs, <a href="#ref-verhoef2018" role="doc-biblioref">Ver Hoef et al. 2018</a>)</span>. Such models have been used repeatedly for modelling spatial autocorrelation in species occupancy and demographic rates <span class="citation" data-cites="saracco2010 saracco2012 guelat2018">(e.g. <a href="#ref-saracco2010" role="doc-biblioref">Saracco et al. 2010</a>, <a href="#ref-saracco2012" role="doc-biblioref">2012</a>; <a href="#ref-guelat2018" role="doc-biblioref">Guélat and Kéry 2018</a>)</span> and are straightforward to implement using NIMBLE <span class="citation" data-cites="lawson2020">(<a href="#ref-lawson2020" role="doc-biblioref">Lawson 2020</a>)</span>. One possible challenge with using CAR models to explicitly model spatial correlations within our ptarmigan IDSM is that CAR models rely on “neighborhood” relationships between discrete areas and many “neighbors” are missing in our ptarmigan data (e.g. <a href="#fig-dens" class="quarto-xref">Figure&nbsp;3</a>). Estimation of latent parameters in missing areas may be possible though (Perry de Valpine, personal communication; <span class="citation" data-cites="schaub2021">Schaub and Kéry (<a href="#ref-schaub2021" role="doc-biblioref">2021</a>)</span> chapter 19), and this may result in a unique opportunity for making predictions of ptarmigan population trends in unmonitored areas, provided that data for a sufficient number and range of areas are available. Here, we may benefit from the fact that the line transect survey data included in this study constitutes just the publicly available part of the data collected through “Hønsefuglportalen” but the programme also includes additional surveys on private land. Extending to data from private land would provide better coverage especially in south-eastern and southern parts of Norway, which includes areas where only very limited amounts of data are collected on public land. Exploring to what degree additional data from Hønsefuglportalen could be included in future studies employing an extended IDSM with additional spatial structuring is therefore a worthwhile endeavor.</p>
<p>Finally, including further data beyond the line transect surveys may be relevant in the future, and in particular in the context of informing and improving management of ptarmigan hunting. In the present study, we have used auxiliary radio-telemetry data to supplement information on survival, but since this data was available for only one out of 41 areas, its influence was small. Nonetheless, this illustrates a way for how smaller datasets from single or subsets of areas can be integrated into a large-scale modelling framework. Other relevant data could be included using the same approach, for example data from ongoing nesting success monitoring, data from past studies of marked birds <span class="citation" data-cites="sandercock_etal2011">(<a href="#ref-sandercock_etal2011" role="doc-biblioref">Sandercock et al. 2011</a>)</span>, and data from other monitoring programs for breeding birds based on point counts (see e.g.&nbsp;the Norwegian Breeding Bird Monitoring: <a href="https://hekkefuglovervakingen.nina.no/" class="uri">https://hekkefuglovervakingen.nina.no/</a>). The most relevant source of data to be included into the IDSM framework in the near future, however, is harvest data. Such data might be available with different spatial and temporal resolutions. First, at the municipality level there are data with national coverage collected annually by Statistics Norway (<a href="https://www.ssb.no/" class="uri">https://www.ssb.no/</a>). Second, many public land owners have data with much higher temporal (daily) and spatial (harvest area) resolution, including both harvest bags and harvest effort (number of hunters per area per day). As the IDSM framework is, in essence, an IPM, harvest can be modelled through partitioning of survival into cause-specific mortality in the process model and inclusion of relevant harvest data likelihoods <span class="citation" data-cites="gamelon2021 nater2021">(e.g., <a href="#ref-gamelon2021" role="doc-biblioref">Gamelon et al. 2021</a>; <a href="#ref-nater2021" role="doc-biblioref">Chloé R. Nater et al. 2021</a>)</span>. While harvest effects on willow ptarmigan have been studied previously, much uncertainty remains <span class="citation" data-cites="sandercock_etal2011 aanes2002 pedersen2004 willebrand2001">(<a href="#ref-sandercock_etal2011" role="doc-biblioref">Sandercock et al. 2011</a>; <a href="#ref-aanes2002" role="doc-biblioref">Aanes et al. 2002</a>; <a href="#ref-pedersen2004" role="doc-biblioref">Pedersen et al. 2004</a>; <a href="#ref-willebrand2001" role="doc-biblioref">Willebrand and Hörnell 2001</a>)</span>. For example, little is known about how harvest pressure and density feedbacks interact on different temporal and spatial scales <span class="citation" data-cites="kvasnes2015">(<a href="#ref-kvasnes2015" role="doc-biblioref">Kvasnes et al. 2015</a>)</span>, despite this knowledge being crucial for preventing over-exploitation and ensuring sustainable harvest <span class="citation" data-cites="williams2002 breisjøberget2018">(<a href="#ref-williams2002" role="doc-biblioref">Williams, Nichols, and Conroy 2002</a>; <a href="#ref-breisjøberget2018" role="doc-biblioref">Breisjøberget, Odden, Storaas, et al. 2018</a>)</span>. Additionally, harvest effects often interact with other (emergent) factors such as climate change and habitat degradation, making predictive models that account for harvest alongside other mechanisms invaluable for informing policy changes <span class="citation" data-cites="gamelon2019">(<a href="#ref-gamelon2019" role="doc-biblioref">Gamelon, Sandercock, and Sæther 2019</a>)</span>.</p>
</section>
<section id="reproducible-workflows-for-a-sustainable-future" class="level2">
<h2 class="anchored" data-anchor-id="reproducible-workflows-for-a-sustainable-future">Reproducible workflows for a sustainable future</h2>
<p>Producing a transparent and reproducible workflow for the analysis presented here was a central objective in this study. We have done this by setting up a well documented, function-based R workflow that allows (re-)running the complete analysis from downloading the publicly available data to visualizing the results produced by the IDSM (<a href="#fig-workflow" class="quarto-xref">Figure&nbsp;2</a>) and that can be implemented using pipelines that employ “R targets” <span class="citation" data-cites="landau2021">(<a href="#ref-landau2021" role="doc-biblioref">Landau 2021</a>)</span> and Nix/GNUparallel <span class="citation" data-cites="nix2004 tange2024">(<a href="#ref-nix2004" role="doc-biblioref">Dolstra, Jonge, and Visser 2004</a>; <a href="#ref-tange2024" role="doc-biblioref">Tange 2024</a>)</span>. Modern applied ecology needs research to be published not just as scientific papers, but as reproducible and well documented workflows <span class="citation" data-cites="lewis2018">(<a href="#ref-lewis2018" role="doc-biblioref">Lewis, Vander Wal, and Fifield 2018</a>)</span>. This is particularly crucial for research that is (to be) closely tied to management and/or used to create biodiversity indicators that are to be reported nationally or internationally, or to be used by industrial partners <span class="citation" data-cites="powers2019">(<a href="#ref-powers2019" role="doc-biblioref">Powers and Hampton 2019</a>)</span>. This is both because of the enhanced transparency and credibility provided by openly available reproducible workflows and because of their cost-effectiveness, which allows for more sustainable use of funding in the mid- to long-term. Finally, open and reproducible workflows facilitate collaboration and inclusion of stakeholders in the research process, paving the path for the translational science that is required for society to tackle the the biodiversity crisis <span class="citation" data-cites="rubert2021">(<a href="#ref-rubert2021" role="doc-biblioref">Rubert-Nason et al. 2021</a>)</span>. It is our hope that this study can serve as an example of where to start.</p>
</section>
</section>
<section id="author-contributions" class="level1">
<h1>Author contributions</h1>
<p><strong>Chloé R. Nater:</strong> Conceptualization, Methodology, Software, Formal analysis, Writing - Original Draft, Writing - Review and editing, Visualization.</p>
<p><strong>Francesco Frassinelli:</strong> Software, Writing - Review and editing.</p>
<p><strong>James A. Martin:</strong> Conceptualization, Writing - Original Draft, Writing - review and editing.</p>
<p><strong>Erlend B. Nilsen:</strong> Conceptualization, Methodology, Data curation, Writing - Original Draft, Writing - Review and editing, Project administration, Funding acquisition</p>
</section>
<section id="acknowledgements" class="level1">
<h1>Acknowledgements</h1>
<p>We thank Mark Hewinson, Todd Arnold, and one anonymous reviewer for constructive comments during the review process. We are grateful to the numerous volunteer observers (and their canine assistants) who collected ptarmigan observations as part of the “Hønsefuglportalen” monitoring effort. Additionally, we would like to thank Bernardo Brandão Niebuhr dos Santos for his help with spatial data visualization. J. A. Martin was supported by a Fulbright Fellowship and the Norwegian Institute for Nature Research, which facilitated this collaboration.</p>
</section>
<section id="funding" class="level1">
<h1>Funding</h1>
<p>Funding for both data collection and analytic work was provided by the Norwegian Environmental Agency (grant numbers 17010522, 19047014, and 22047004).</p>
</section>
<section id="conflict-of-interest-disclosure" class="level1">
<h1>Conflict of interest disclosure</h1>
<p>The authors declare that they comply with the PCI rule of having no financial conflicts of interest in relation to the content of the article.</p>
</section>
<section id="data-and-code-availability" class="level1">
<h1>Data and code availability</h1>
<p>The raw data from the line transect surveys is deposited on GBIF and can be accessed freely via the Living Norway Data Portal (https://data.livingnorway.no/). The work presented above is based on versions 1.7, 1.8, and 1.12 for the datasets from Fjellstyrene <span class="citation" data-cites="fjellstyre2022">(<a href="#ref-fjellstyre2022" role="doc-biblioref">E. B. Nilsen, Vang, Kjønsberg, and J. 2022</a>)</span>, Statskog <span class="citation" data-cites="statskog2022">(<a href="#ref-statskog2022" role="doc-biblioref">E. B. Nilsen, Vang, and I. 2022</a>)</span>, and FeFo <span class="citation" data-cites="fefo2022">(<a href="#ref-fefo2022" role="doc-biblioref">E. B. Nilsen, Vang, Kjønsberg, and E. 2022</a>)</span>, respectively.</p>
<p>The auxiliary radio-telemetry data, rodent occupancy data, posterior summaries, and supplementary figures are archived on OSF <span class="citation" data-cites="Nater_Nilsen_Martin_2024">(<a href="#ref-Nater_Nilsen_Martin_2024" role="doc-biblioref">Chloé R. Nater, Nilsen, and Martin 2024</a>)</span>.</p>
<p>All code, including the three pipelines, can be found in the project’s repository on GitHub: <a href="https://github.com/ErlendNilsen/OpenPop_Integrated_DistSamp" class="uri">https://github.com/ErlendNilsen/OpenPop_Integrated_DistSamp</a>. The results presented in this paper were created using version 2.1 of the code <span class="citation" data-cites="rypeIDSM2024">(<a href="#ref-rypeIDSM2024" role="doc-biblioref">Chloé R. Nater et al. 2024</a>)</span>.</p>
</section>
<section id="references" class="level1 unnumbered">


</section>

<div id="quarto-appendix" class="default"><section class="quarto-appendix-contents" role="doc-bibliography" id="quarto-bibliography"><h2 class="anchored quarto-appendix-heading">References</h2><div id="refs" class="references csl-bib-body hanging-indent" data-entry-spacing="0" role="list">
<div id="ref-aanes2002" class="csl-entry" role="listitem">
Aanes, Sondre, Steinar Engen, Bernt-Erik SÆther, Tomas Willebrand, and Vidar Marcström. 2002. <span>“Sustainable Harvesting Strategies of Willow Ptarmigan in a Fluctuating Environment.”</span> <em>Ecological Applications</em> 12 (1): 281–90.
</div>
<div id="ref-aebischer2010" class="csl-entry" role="listitem">
Aebischer, Nicholas J., and Julie A. Ewald. 2010. <span>“Grey Partridge Perdix Perdix in the UK: Recovery Status, Set-Aside and Shooting.”</span> <em>Ibis</em> 152 (3): 530–42. https://doi.org/<a href="https://doi.org/10.1111/j.1474-919X.2010.01037.x">https://doi.org/10.1111/j.1474-919X.2010.01037.x</a>.
</div>
<div id="ref-alice2016" class="csl-entry" role="listitem">
Alice Boyle, W, Brett K Sandercock, and Kathy Martin. 2016. <span>“Patterns and Drivers of Intraspecific Variation in Avian Life History Along Elevational Gradients: A Meta-Analysis.”</span> <em>Journal of Animal Ecology</em> 91 (2): 469–82.
</div>
<div id="ref-andrewartha1954" class="csl-entry" role="listitem">
Andrewartha, Herbert George, and L. Charles Birch. 1954. <em>The Distribution and Abundance of Animals.</em> University of Chicago press.
</div>
<div id="ref-Arnekleiv2022" class="csl-entry" role="listitem">
Arnekleiv, Øyvind, Katrine Eldegard, Pål F. Moa, Lasse F. Eriksen, and Erlend B. Nilsen. 2022. <span>“Drivers and Consequences of Partial Migration in an Alpine Bird Species.”</span> Journal Article. <em>Ecology and Evolution</em> 12 (3): e8690. https://doi.org/<a href="https://doi.org/10.1002/ece3.8690">https://doi.org/10.1002/ece3.8690</a>.
</div>
<div id="ref-bears2009" class="csl-entry" role="listitem">
Bears, H, K Martin, and GC White. 2009. <span>“Breeding in High-Elevation Habitat Results in Shift to Slower Life-History Strategy Within a Single Species.”</span> <em>Journal of Animal Ecology</em> 78 (2): 365–75.
</div>
<div id="ref-bond2021" class="csl-entry" role="listitem">
Bond, Monica L, Barbara König, Arpat Ozgul, Damien R Farine, and Derek E Lee. 2021. <span>“Socially Defined Subpopulations Reveal Demographic Variation in a Giraffe Metapopulation.”</span> <em>The Journal of Wildlife Management</em> 85 (5): 920–31.
</div>
<div id="ref-bowler2020" class="csl-entry" role="listitem">
Bowler, Diana E, Mikkel AJ Kvasnes, Hans C Pedersen, Brett K Sandercock, and Erlend B Nilsen. 2020. <span>“Impacts of Predator-Mediated Interactions Along a Climatic Gradient on the Population Dynamics of an Alpine Bird.”</span> <em>Proceedings of the Royal Society B</em> 287 (1941): 20202653.
</div>
<div id="ref-breisjøberget2018" class="csl-entry" role="listitem">
Breisjøberget, Jo Inge, Morten Odden, Torstein Storaas, Erlend B. Nilsen, and Mikkel A. J. Kvasnes. 2018. <span>“Harvesting a Red-Listed Species: Determinant Factors for Willow Ptarmigan Harvest Rates, Bag Sizes, and Hunting Efforts in Norway.”</span> <em>European Journal of Wildlife Research</em> 64 (5): 54. <a href="https://doi.org/10.1007/s10344-018-1208-8">https://doi.org/10.1007/s10344-018-1208-8</a>.
</div>
<div id="ref-Breisjøberget_etal_2018" class="csl-entry" role="listitem">
Breisjøberget, Jo Inge, Morten Odden, Per Wegge, Barbara Zimmermann, and Harry Andreassen. 2018. <span>“The Alternative Prey Hypothesis Revisited: Still Valid for Willow Ptarmigan Population Dynamics.”</span> <em>PLOS ONE</em> 13 (6): 1–14. <a href="https://doi.org/10.1371/journal.pone.0197289">https://doi.org/10.1371/journal.pone.0197289</a>.
</div>
<div id="ref-bro2003" class="csl-entry" role="listitem">
Bro, Elisabeth, Benoît Deldalle, Manuel Massot, François Reitz, and Slaheddine Selmi. 2003. <span>“Density Dependence of Reproductive Success in Grey Partridge Perdix Perdix Populations in France: Management Implications.”</span> <em>Wildlife Biology</em> 9 (2): 93–102. <a href="https://doi.org/10.2981/wlb.2003.031">https://doi.org/10.2981/wlb.2003.031</a>.
</div>
<div id="ref-brooks2001" class="csl-entry" role="listitem">
Brooks, Elizabeth N, and Jean-Dominique Lebreton. 2001. <span>“Optimizing Removals to Control a Metapopulation: Application to the Yellow Legged Herring Gull (Larus Cachinnans).”</span> <em>Ecological Modelling</em> 136 (2-3): 269–84.
</div>
<div id="ref-caswell2000" class="csl-entry" role="listitem">
Caswell, Hal. 2000. <em>Matrix Population Models</em>. Vol. 1. Sinauer Sunderland, MA.
</div>
<div id="ref-christie2020" class="csl-entry" role="listitem">
Christie, Alec P, Tatsuya Amano, Philip A Martin, Silviu O Petrovan, Gorm E Shackelford, Benno I Simmons, Rebecca K Smith, David R Williams, Claire FR Wordley, and William J Sutherland. 2020. <span>“Poor Availability of Context-Specific Evidence Hampers Decision-Making in Conservation.”</span> <em>Biological Conservation</em> 248: 108666.
</div>
<div id="ref-dickinson2010" class="csl-entry" role="listitem">
Dickinson, Janis L, Benjamin Zuckerberg, and David N Bonter. 2010. <span>“Citizen Science as an Ecological Research Tool: Challenges and Benefits.”</span> <em>Annual Review of Ecology, Evolution, and Systematics</em> 41: 149–72.
</div>
<div id="ref-nix2004" class="csl-entry" role="listitem">
Dolstra, Eelco, Merijn de Jonge, and Eelco Visser. 2004. <span>“Nix: A Safe and Policy-Free System for Software Deployment.”</span> In <em>Proceedings of the 18th USENIX Conference on System Administration</em>, 79–92. LISA ’04. USA: USENIX Association.
</div>
<div id="ref-Elton1942" class="csl-entry" role="listitem">
Elton, Charles. 1942. <em>Voles, Mice and Lemmings: Problems in Population Dynamics</em>. Clarendon Press.
</div>
<div id="ref-Eriksen2018" class="csl-entry" role="listitem">
Eriksen, L. F., P. F. Moa, and E. B. Nilsen. 2018. <span>“Quantifying Risk of Overharvest When Implementation Is Uncertain.”</span> Journal Article. <em>Journal of Applied Ecology</em> 55 (2): 482–93. <a href="https://doi.org/10.1111/1365-2664.12992">https://doi.org/10.1111/1365-2664.12992</a>.
</div>
<div id="ref-Eriksen2023" class="csl-entry" role="listitem">
Eriksen, L. F., T. H. Ringsby, H. C. Pedersen, and E. B. Nilsen. 2023. <span>“Climatic Forcing and Individual Heterogeneity in a Resident Mountain Bird: Legacy Data Reveal Effects on Reproductive Strategies.”</span> Journal Article. <em>Royal Society Open Science</em> 10 (5): 221427. <a href="https://doi.org/doi:10.1098/rsos.221427">https://doi.org/doi:10.1098/rsos.221427</a>.
</div>
<div id="ref-feld2009" class="csl-entry" role="listitem">
Feld, Christian K, Pedro Martins da Silva, José Paulo Sousa, Francesco De Bello, Rob Bugter, Ulf Grandin, Daniel Hering, et al. 2009. <span>“Indicators of Biodiversity and Ecosystem Services: A Synthesis Across Ecosystems and Spatial Scales.”</span> <em>Oikos</em> 118 (12): 1862–71.
</div>
<div id="ref-fraisl2022" class="csl-entry" role="listitem">
Fraisl, Dilek, Gerid Hager, Baptiste Bedessem, Margaret Gold, Pen-Yuan Hsing, Finn Danielsen, Colleen B Hitchcock, et al. 2022. <span>“Citizen Science in Environmental and Ecological Sciences.”</span> <em>Nature Reviews Methods Primers</em> 2 (1): 64.
</div>
<div id="ref-framstad2022" class="csl-entry" role="listitem">
Framstad, Erik, Nina E Eide, Wenche Eide, Kari Klanderud, Anders Kolstad, Joachim Töpper, and Vigdis Vandvik. 2022. <span>“Vurdering Av <span class="nocase">ø</span>kologisk Tilstand for Fjell i Norge i 2021.”</span> <em>NINA Rapport</em> 2050.
</div>
<div id="ref-Alastair2020" class="csl-entry" role="listitem">
Franke, Alastair, Knud Falk, Kevin Hawkshaw, Skip Ambrose, David L. Anderson, Peter J. Bente, Travis Booms, et al. 2020. <span>“Status and Trends of Circumpolar Peregrine Falcon and Gyrfalcon Populations.”</span> Journal Article. <em>Ambio</em> 49 (3): 762–83. <a href="https://doi.org/10.1007/s13280-019-01300-z">https://doi.org/10.1007/s13280-019-01300-z</a>.
</div>
<div id="ref-Freckleton2006" class="csl-entry" role="listitem">
Freckleton, R. P., A. R. Watkinson, R. E. Green, and W. J. Sutherland. 2006. <span>“Census Error and the Detection of Density Dependence.”</span> Journal Article. <em>Journal of Animal Ecology</em> 75 (4): 837–51. <a href="https://doi.org/10.1111/j.1365-2656.2006.01121.x">https://doi.org/10.1111/j.1365-2656.2006.01121.x</a>.
</div>
<div id="ref-fuglei2020" class="csl-entry" role="listitem">
Fuglei, Eva, John-André Henden, Chris T. Callahan, Olivier Gilg, Jannik Hansen, Rolf A. Ims, Arkady P. Isaev, et al. 2020. <span>“Circumpolar Status of Arctic Ptarmigan: Population Dynamics and Trends.”</span> <em>Ambio</em> 49 (3): 749–61. <a href="https://doi.org/10.1007/s13280-019-01191-0">https://doi.org/10.1007/s13280-019-01191-0</a>.
</div>
<div id="ref-gamelon2021" class="csl-entry" role="listitem">
Gamelon, Marlène, Chloé R Nater, Éric Baubet, Aurélien Besnard, Laura Touzot, Jean-Michel Gaillard, Jean-dominique Lebreton, and Olivier Gimenez. 2021. <span>“Efficient Use of Harvest Data: A Size-Class-Structured Integrated Population Model for Exploited Populations.”</span> <em>Ecography</em> 44 (9): 1296–1310.
</div>
<div id="ref-gamelon2019" class="csl-entry" role="listitem">
Gamelon, Marlène, Brett K. Sandercock, and Bernt-Erik Sæther. 2019. <span>“Does Harvesting Amplify Environmentally Induced Population Fluctuations over Time in Marine and Terrestrial Species?”</span> <em>Journal of Applied Ecology</em> 56 (9): 2186–94. <a href="https://doi.org/10.1111/1365-2664.13466">https://doi.org/10.1111/1365-2664.13466</a>.
</div>
<div id="ref-guelat2018" class="csl-entry" role="listitem">
Guélat, Jérôme, and Marc Kéry. 2018. <span>“Effects of Spatial Autocorrelation and Imperfect Detection on Species Distribution Models.”</span> <em>Methods in Ecology and Evolution</em> 9 (6): 1614–25.
</div>
<div id="ref-hagen1952" class="csl-entry" role="listitem">
Hagen, Yngvar. 1952. <em>Rovfuglene Og Viltpleien</em>. Gyldendal Norsk forlag.
</div>
<div id="ref-hamilton2018" class="csl-entry" role="listitem">
Hamilton, Olivia NP, Sophie E Kincaid, Rochelle Constantine, Lily Kozmian-Ledward, Cameron G Walker, and Rachel M Fewster. 2018. <span>“Accounting for Uncertainty in Duplicate Identification and Group Size Judgements in Mark–Recapture Distance Sampling.”</span> <em>Methods in Ecology and Evolution</em> 9 (2): 354–62.
</div>
<div id="ref-Henden2020" class="csl-entry" role="listitem">
Henden, J. A., R. A. Ims, N. G. Yoccoz, E. J. Asbjornsen, A. Stien, J. P. Mellard, T. Tveraa, F. Marolla, and J. U. Jepsen. 2020. <span>“End-User Involvement to Improve Predictions and Management of Populations with Complex Dynamics and Multiple Drivers.”</span> Journal Article. <em>Ecol Appl</em> 30 (6): e02120. <a href="https://doi.org/10.1002/eap.2120">https://doi.org/10.1002/eap.2120</a>.
</div>
<div id="ref-Henden2017" class="csl-entry" role="listitem">
Henden, John-André, Rolf Anker Ims, Eva Fuglei, and Åshild Ønvik Pedersen. 2017. <span>“Changed Arctic-Alpine Food Web Interactions Under Rapid Climate Warming: Implication for Ptarmigan Research”</span> 2017 (SP1): wlb.00240. https://doi.org/<a href="https://doi.org/10.2981/wlb.00240">https://doi.org/10.2981/wlb.00240</a>.
</div>
<div id="ref-hjeljord2022" class="csl-entry" role="listitem">
Hjeljord, Olav, and Leif Egil Loe. 2022. <span>“The Roles of Climate and Alternative Prey in Explaining 142 Years of Declining Willow Ptarmigan Hunting Yield.”</span> <em>Wildlie Biology</em> 2022 (6): e01058. https://doi.org/<a href="https://doi.org/10.1002/wlb3.01058">https://doi.org/10.1002/wlb3.01058</a>.
</div>
<div id="ref-horswill2019" class="csl-entry" role="listitem">
Horswill, Cat, Holly K Kindsvater, Maria José Juan-Jordá, Nicholas K Dulvy, Marc Mangel, and Jason Matthiopoulos. 2019. <span>“Global Reconstruction of Life-History Strategies: A Case Study Using Tunas.”</span> <em>Journal of Applied Ecology</em> 56 (4): 855–65.
</div>
<div id="ref-horswill2021" class="csl-entry" role="listitem">
Horswill, Cat, Andrea Manica, Francis Daunt, Mark Newell, Sarah Wanless, Matthew Wood, and Jason Matthiopoulos. 2021. <span>“Improving Assessments of Data-Limited Populations Using Life-History Theory.”</span> <em>Journal of Applied Ecology</em> 58 (6): 1225–36.
</div>
<div id="ref-Israelsen2020" class="csl-entry" role="listitem">
Israelsen, M. F., L. F. Eriksen, P. F. Moa, B. R. Hagen, and E. B. Nilsen. 2020. <span>“Survival and Cause-Specific Mortality of Harvested Willow Ptarmigan (Lagopus Lagopus) in Central Norway.”</span> <em>Ecol Evol</em> 10 (20): 11144–54. <a href="https://doi.org/10.1002/ece3.6754">https://doi.org/10.1002/ece3.6754</a>.
</div>
<div id="ref-jakobsson2020" class="csl-entry" role="listitem">
Jakobsson, Simon, and Bård Pedersen. 2020. <span>“Naturindeks for Norge 2020. Tilstand Og Utvikling for Biologisk Mangfold.”</span> <em>NINA Rapport</em> 1886.
</div>
<div id="ref-jetz2019" class="csl-entry" role="listitem">
Jetz, Walter, Melodie A McGeoch, Robert Guralnick, Simon Ferrier, Jan Beck, Mark J Costello, Miguel Fernandez, et al. 2019. <span>“Essential Biodiversity Variables for Mapping and Monitoring Species Populations.”</span> <em>Nature Ecology &amp; Evolution</em> 3 (4): 539–51.
</div>
<div id="ref-johnston2023" class="csl-entry" role="listitem">
Johnston, Alison, Eleni Matechou, and Emily B Dennis. 2023. <span>“Outstanding Challenges and Future Directions for Biodiversity Monitoring Using Citizen Science Data.”</span> <em>Methods in Ecology and Evolution</em> 14 (1): 103–16.
</div>
<div id="ref-Kjellander_Nordström_2003" class="csl-entry" role="listitem">
Kjellander, Petter, and Jonas Nordström. 2003. <span>“Cyclic Voles, Prey Switching in Red Fox, and Roe Deer Dynamics – a Test of the Alternative Prey Hypothesis.”</span> <em>Oikos</em> 101 (2): 338–44. <a href="https://doi.org/10.1034/j.1600-0706.2003.11986.x">https://doi.org/10.1034/j.1600-0706.2003.11986.x</a>.
</div>
<div id="ref-kvasnes2015" class="csl-entry" role="listitem">
Kvasnes, Mikkel A. J., Hans Chr. Pedersen, Håkon Solvang, Torstein Storaas, and Erlend B. Nilsen. 2015. <span>“Spatial Distribution and Settlement Strategies in Willow Ptarmigan.”</span> <em>Population Ecology</em> 57 (1): 151–61. <a href="https://doi.org/10.1007/s10144-014-0454-1">https://doi.org/10.1007/s10144-014-0454-1</a>.
</div>
<div id="ref-landau2021" class="csl-entry" role="listitem">
Landau, William Michael. 2021. <span>“The Targets r Package: A Dynamic Make-Like Function-Oriented Pipeline Toolkit for Reproducibility and High-Performance Computing.”</span> <em>Journal of Open Source Software</em> 6 (57): 2959. <a href="https://doi.org/10.21105/joss.02959">https://doi.org/10.21105/joss.02959</a>.
</div>
<div id="ref-lawson2020" class="csl-entry" role="listitem">
Lawson, Andrew B. 2020. <span>“NIMBLE for Bayesian Disease Mapping.”</span> <em>Spatial and Spatio-Temporal Epidemiology</em> 33: 100323.
</div>
<div id="ref-lewis2018" class="csl-entry" role="listitem">
Lewis, Keith P, Eric Vander Wal, and David A Fifield. 2018. <span>“Wildlife Biology, Big Data, and Reproducible Research.”</span> <em>Wildlife Society Bulletin</em> 42 (1): 172–79.
</div>
<div id="ref-Linden1988" class="csl-entry" role="listitem">
Linden, H. 1988. <span>“Latitudinal Gradients in Predator-Prey Interactions, Cyclicity and Synchronism in Voles and Small Game Populations in Finland.”</span> Journal Article. <em>Oikos</em> 52 (3): 341–49. <a href="<Go to ISI>://A1988N893200014 ">&lt;Go to ISI&gt;://A1988N893200014 </a>.
</div>
<div id="ref-marques2004" class="csl-entry" role="listitem">
Marques, Tiago A. 2004. <span>“Predicting and Correcting Bias Caused by Measurement Error in Line Transect Sampling Using Multiplicative Error Models.”</span> <em>Biometrics</em> 60 (3): 757–63.
</div>
<div id="ref-mcconnell2018" class="csl-entry" role="listitem">
McConnell, Mark D., Adrian P. Monroe, Richard Chandler, William E. Palmer, Shane D. Wellendorf, Jr., L. Wes Burger, and James A. Martin. 2018. <span>“Factors Influencing Northern Bobwhite Recruitment, with Implications for Population Growth.”</span> <em>The Auk</em> 135 (4): 1087–99. <a href="https://doi.org/10.1642/AUK-18-49.1">https://doi.org/10.1642/AUK-18-49.1</a>.
</div>
<div id="ref-mccrea2013" class="csl-entry" role="listitem">
McCrea, Rachel S, Byron JT Morgan, and Diana J Cole. 2013. <span>“Age-Dependent Mixture Models for Recovery Data on Animals Marked at Unknown Age.”</span> <em>Journal of the Royal Statistical Society Series C: Applied Statistics</em> 62 (1): 101–13.
</div>
<div id="ref-mcghee2007" class="csl-entry" role="listitem">
McGhee, Jay D., and James M. Berkson. 2007. <span>“Estimation of a Nonlinear Density-Dependence Parameter for Wild Turkey.”</span> <em>Journal of Wildlife Management</em> 71 (3): 706–12. <a href="https://doi.org/10.2193/2005-630">https://doi.org/10.2193/2005-630</a>.
</div>
<div id="ref-morrison2022" class="csl-entry" role="listitem">
Morrison, Catriona A, Simon J Butler, Jacquie A Clark, Juan Arizaga, Oriol Baltà, Jaroslav Cepák, Arantza Leal Nebot, et al. 2022. <span>“Demographic Variation in Space and Time: Implications for Conservation Targeting.”</span> <em>Royal Society Open Science</em> 9 (3): 211671.
</div>
<div id="ref-myrberget1988" class="csl-entry" role="listitem">
Myrberget, S. 1988. <span>“Demography of an Island Population of Willow Ptarmigan in Northern Norway.<span></span>s. 379-419. I Bergerud, a. T, &amp; Gratson, MW [Red.], Adaptive Strategies and Population Ecology of Northern Grouse.”</span> <em>I. Population Studies. University of Minnesota Press. Minneapolis, Minnesota, USA</em>.
</div>
<div id="ref-nakagawa2013" class="csl-entry" role="listitem">
Nakagawa, Shinichi, and Holger Schielzeth. 2013. <span>“A General and Simple Method for Obtaining R2 from Generalized Linear Mixed-Effects Models.”</span> <em>Methods in Ecology and Evolution</em> 4 (2): 133–42.
</div>
<div id="ref-nater2023" class="csl-entry" role="listitem">
Nater, Chloé R, Malcolm D Burgess, Peter Coffey, Bob Harris, Frank Lander, David Price, Mike Reed, and Robert A Robinson. 2023. <span>“Spatial Consistency in Drivers of Population Dynamics of a Declining Migratory Bird.”</span> <em>Journal of Animal Ecology</em> 92 (1): 97–111.
</div>
<div id="ref-nater2021" class="csl-entry" role="listitem">
Nater, Chloé R, Nina E Eide, Åshild Ø Pedersen, Nigel G Yoccoz, and Eva Fuglei. 2021. <span>“Contributions from Terrestrial and Marine Resources Stabilize Predator Populations in a Rapidly Changing Climate.”</span> <em>Ecosphere</em> 12 (6): e03546.
</div>
<div id="ref-rypeIDSM2024" class="csl-entry" role="listitem">
Nater, Chloé R., ErlendNilsen, christofferhohi, Matthew Grainger, Bernardo Brandão Niebuhr, and Francesco Frassinelli. 2024. <span>“<span class="nocase">ErlendNilsen/OpenPop_Integrated_DistSamp: Ptarmigan IDSM v2.1</span>.”</span> Zenodo. <a href="https://doi.org/10.5281/zenodo.13767267">https://doi.org/10.5281/zenodo.13767267</a>.
</div>
<div id="ref-Nater_Nilsen_Martin_2024" class="csl-entry" role="listitem">
Nater, Chloé R, Erlend B Nilsen, and James Martin. 2024. <span>“Large-Scale Spatiotemporal Variation in Vital Rates and Population Dynamics of an Alpine Bird.”</span> OSF. <a href="https://doi.org/10.17605/OSF.IO/7326R">https://doi.org/10.17605/OSF.IO/7326R</a>.
</div>
<div id="ref-nater2018" class="csl-entry" role="listitem">
Nater, Chloé R, Atle Rustadbakken, Torbjørn Ergon, Øystein Langangen, S Jannicke Moe, Yngvild Vindenes, Leif Asbjørn Vøllestad, and Per Aass. 2018. <span>“Individual Heterogeneity and Early Life Conditions Shape Growth in a Freshwater Top Predator.”</span> <em>Ecology</em> 99 (5): 1011–17.
</div>
<div id="ref-nilsen2023" class="csl-entry" role="listitem">
Nilsen, E. B., and C. R. Nater. 2024. <span>“An Integrated Open Population Distance Sampling Approach for Modelling Age-Structured Populations.”</span> <em>EcoEvoRxiv</em>. <a href="https://doi.org/10.32942/X2Q899">https://doi.org/10.32942/X2Q899</a>.
</div>
<div id="ref-nilsen2020" class="csl-entry" role="listitem">
Nilsen, E. B., and L. Rød-Eriksen. 2020. <span>“Trender i St<span>ø</span>rrelsen p<span>å</span> Den Norske Lirypebestanden i Perioden 2009-2020. Analyser Basert p<span>å</span> Data Fra h<span>ø</span>nsefuglportalen.”</span> <em>NINA Rapport</em> 1869.
</div>
<div id="ref-statskog2022" class="csl-entry" role="listitem">
Nilsen, E. B., R. Vang, and Breisjøberget J. I. 2022. <span>“Tetraonid Line Transect Surveys from Norway: Data from Statskog. Version 1.8.”</span> Norwegian Institute for Nature Research. Sampling event dataset. <a href="https://doi.org/10.15468/q2ehlk">https://doi.org/10.15468/q2ehlk</a>.
</div>
<div id="ref-fefo2022" class="csl-entry" role="listitem">
Nilsen, E. B., R. Vang, M. Kjønsberg, and Asbjørnsen E. 2022. <span>“Tetraonid Line Transect Surveys from Norway: Data from Finnmarkseiendommen (FeFo). Version 1.12.”</span> Norwegian Institute for Nature Research. Sampling event dataset. <a href="https://doi.org/10.15468/s7c8qd">https://doi.org/10.15468/s7c8qd</a>.
</div>
<div id="ref-fjellstyre2022" class="csl-entry" role="listitem">
Nilsen, E. B., R. Vang, M. Kjønsberg, and Kvasnes M. A. J. 2022. <span>“Tetraonid Line Transect Surveys from Norway: Data from Fjellstyrene. Version 1.7.”</span> Norwegian Institute for Nature Research. Sampling event dataset. <a href="https://doi.org/10.15468/975ski">https://doi.org/10.15468/975ski</a>.
</div>
<div id="ref-nilsen2009" class="csl-entry" role="listitem">
Nilsen, Erlend B, Jean-Michel Gaillard, Reidar Andersen, John Odden, Daniel Delorme, Guy Van Laere, and John DC Linnell. 2009. <span>“A Slow Life in Hell or a Fast Life in Heaven: Demographic Analyses of Contrasting Roe Deer Populations.”</span> <em>Journal of Animal Ecology</em> 78 (3): 585–94.
</div>
<div id="ref-Novoa2016" class="csl-entry" role="listitem">
Novoa, C., G. Astruc, J. F. Desmet, and A. Besnard. 2016. <span>“No Short-Term Effects of Climate Change on the Breeding of Rock Ptarmigan in the French Alps and Pyrenees.”</span> Journal Article. <em>Journal of Ornithology</em> 157 (3): 797–810. <a href="https://doi.org/10.1007/s10336-016-1335-5">https://doi.org/10.1007/s10336-016-1335-5</a>.
</div>
<div id="ref-Nyström_etal_2006" class="csl-entry" role="listitem">
Nyström, J., L. Dalén, P. Hellström, J. Ekenstedt, H. Angleby, and A. Angerbjörn. 2006. <span>“Effect of Local Prey Availability on Gyrfalcon Diet: DNA Analysis on Ptarmigan Remains at Nest Sites.”</span> <em>Journal of Zoology</em> 269 (1): 57–64. https://doi.org/<a href="https://doi.org/10.1111/j.1469-7998.2006.00050.x">https://doi.org/10.1111/j.1469-7998.2006.00050.x</a>.
</div>
<div id="ref-pacifici2017" class="csl-entry" role="listitem">
Pacifici, Krishna, Brian J Reich, David AW Miller, Beth Gardner, Glenn Stauffer, Susheela Singh, Alexa McKerrow, and Jaime A Collazo. 2017. <span>“Integrating Multiple Data Sources in Species Distribution Modeling: A Framework for Data Fusion.”</span> <em>Ecology</em> 98 (3): 840–50.
</div>
<div id="ref-pedersen2004" class="csl-entry" role="listitem">
Pedersen, HC, H Steen, L Kastdalen, H Brøseth, RA Ims, W Svendsen, and NG Yoccoz. 2004. <span>“Weak Compensation of Harvest Despite Strong Density–Dependent Growth in Willow Ptarmigan.”</span> <em>Proceedings of the Royal Society of London. Series B: Biological Sciences</em> 271 (1537): 381–85.
</div>
<div id="ref-pereira2013" class="csl-entry" role="listitem">
Pereira, Henrique Miguel, Simon Ferrier, Michele Walters, Gary N Geller, Rob HG Jongman, Robert J Scholes, Michael William Bruford, et al. 2013. <span>“Essential Biodiversity Variables.”</span> <em>Science</em> 339 (6117): 277–78.
</div>
<div id="ref-powers2019" class="csl-entry" role="listitem">
Powers, Stephen M, and Stephanie E Hampton. 2019. <span>“Open Science, Reproducibility, and Transparency in Ecology.”</span> <em>Ecological Applications</em> 29 (1): e01822.
</div>
<div id="ref-proencca2017" class="csl-entry" role="listitem">
Proença, Vânia, Laura Jane Martin, Henrique Miguel Pereira, Miguel Fernandez, Louise McRae, Jayne Belnap, Monika Böhm, et al. 2017. <span>“Global Biodiversity Monitoring: From Data Sources to Essential Biodiversity Variables.”</span> <em>Biological Conservation</em> 213: 256–63.
</div>
<div id="ref-R2024" class="csl-entry" role="listitem">
R Core Team. 2024. <em>R: A Language and Environment for Statistical Computing</em>. Vienna, Austria: R Foundation for Statistical Computing. <a href="https://www.R-project.org/">https://www.R-project.org/</a>.
</div>
<div id="ref-Reif_Tornberg_Jungell_Korpimäki_2001" class="csl-entry" role="listitem">
Reif, Vitali, Risto Tornberg, Sven Jungell, and Erkki Korpimäki. 2001. <span>“Diet Variation of Common Buzzards in Finland Supports the Alternative Prey Hypothesis.”</span> <em>Ecography</em> 24 (3): 267–74. <a href="https://doi.org/10.1034/j.1600-0587.2001.240304.x">https://doi.org/10.1034/j.1600-0587.2001.240304.x</a>.
</div>
<div id="ref-robinson2014" class="csl-entry" role="listitem">
Robinson, Robert A, Catriona A Morrison, and Stephen R Baillie. 2014. <span>“Integrating Demographic Data: Towards a Framework for Monitoring Wildlife Populations at Large Spatial Scales.”</span> <em>Methods in Ecology and Evolution</em> 5 (12): 1361–72.
</div>
<div id="ref-rubert2021" class="csl-entry" role="listitem">
Rubert-Nason, Kennedy, AM Aramati Casper, Matt Jurjonas, Caitlin Mandeville, Rebecca Potter, and Kirsten Schwarz. 2021. <span>“Ecologist Engagement in Translational Science Is Imperative for Building Resilience to Global Change Threats.”</span> <em>Rethinking Ecology</em> 6: 65–92.
</div>
<div id="ref-sandercock2005" class="csl-entry" role="listitem">
Sandercock, Brett K., Kathy Martin, and Susan J Hannon. 2005. <span>“Life History Strategies in Extreme Environments: Comparative Demography of Arctic and Alpine Ptarmigan.”</span> <em>Ecology</em> 86 (8): 2176–86.
</div>
<div id="ref-sandercock_etal2011" class="csl-entry" role="listitem">
Sandercock, Brett K., Erlend B. Nilsen, Henrik Brøseth, and Hans C. Pedersen. 2011. <span>“Is Hunting Mortality Additive or Compensatory to Natural Mortality? Effects of Experimental Harvest on the Survival and Cause-Specific Mortality of Willow Ptarmigan.”</span> <em>Journal of Animal Ecology</em> 80 (1): 244–58. https://doi.org/<a href="https://doi.org/10.1111/j.1365-2656.2010.01769.x">https://doi.org/10.1111/j.1365-2656.2010.01769.x</a>.
</div>
<div id="ref-saracco2010" class="csl-entry" role="listitem">
Saracco, James F, J Andrew Royle, David F DeSante, and Beth Gardner. 2010. <span>“Modeling Spatial Variation in Avian Survival and Residency Probabilities.”</span> <em>Ecology</em> 91 (7): 1885–91.
</div>
<div id="ref-saracco2012" class="csl-entry" role="listitem">
———. 2012. <span>“Spatial Modeling of Survival and Residency and Application to the Monitoring Avian Productivity and Survivorship Program.”</span> <em>Journal of Ornithology</em> 152: 469–76.
</div>
<div id="ref-schaub2021" class="csl-entry" role="listitem">
Schaub, Michael, and Marc Kéry. 2021. <em>Integrated Population Models: Theory and Ecological Applications with r and JAGS</em>. Academic Press.
</div>
<div id="ref-schmeller2018" class="csl-entry" role="listitem">
Schmeller, Dirk S, Lauren V Weatherdon, Adeline Loyau, Alberte Bondeau, Lluis Brotons, Neil Brummitt, Ilse R Geijzendorffer, et al. 2018. <span>“A Suite of Essential Biodiversity Variables for Detecting Critical Biodiversity Change.”</span> <em>Biological Reviews</em> 93 (1): 55–71.
</div>
<div id="ref-stearns1992" class="csl-entry" role="listitem">
Stearns, Stephen C. 1992. <em>The Evolution of Life Histories</em>. Vol. 249. Oxford university press Oxford.
</div>
<div id="ref-Steen1996" class="csl-entry" role="listitem">
Steen, H., and K. E. Erikstad. 1996. <span>“Sensivity of Willow Grouse Lagopus Lagopus Population Dynamics to Variations in Demographic Parameters.”</span> Journal Article. <em>Wildlife Biology</em> 2: 27–35.
</div>
<div id="ref-Steen1988" class="csl-entry" role="listitem">
Steen, J. B., H. Steen, N. C. Stenseth, S. Myrberget, and V. Marcström. 1988. <span>“Microtine Density and Weather as Predictors of Chick Production in Willow Ptarmigan, Lagopus Lagopus.”</span> Journal Article. <em>Oikos</em> 51 (3): 367–73.
</div>
<div id="ref-stevenson2021" class="csl-entry" role="listitem">
Stevenson, Simone L, Kate Watermeyer, Giovanni Caggiano, Elizabeth A Fulton, Simon Ferrier, and Emily Nicholson. 2021. <span>“Matching Biodiversity Indicators to Policy Needs.”</span> <em>Conservation Biology</em> 35 (2): 522–32.
</div>
<div id="ref-Storch2007" class="csl-entry" role="listitem">
Storch, Ilse. 2007. <span>“Conservation Status of Grouse Worldwide: An Update.”</span> Journal Article. <em>Wildlife Biology</em> 13 (sp1): 5–12. <a href="https://doi.org/10.2981/0909-6396(2007)13[5:csogwa]2.0.co;2">https://doi.org/10.2981/0909-6396(2007)13[5:csogwa]2.0.co;2</a>.
</div>
<div id="ref-tange2024" class="csl-entry" role="listitem">
Tange, Ole. 2024. <span>“GNU Parallel 20240422 (’Børsen’).”</span> Zenodo. <a href="https://doi.org/10.5281/zenodo.11043435">https://doi.org/10.5281/zenodo.11043435</a>.
</div>
<div id="ref-valpine2017" class="csl-entry" role="listitem">
Valpine, Perry de, Daniel Turek, Christopher J Paciorek, Clifford Anderson-Bergman, Duncan Temple Lang, and Rastislav Bodik. 2017. <span>“Programming with Models: Writing Statistical Algorithms for General Model Structures with NIMBLE.”</span> <em>Journal of Computational and Graphical Statistics</em> 26 (2): 403–13.
</div>
<div id="ref-verhoef2018" class="csl-entry" role="listitem">
Ver Hoef, Jay M, Erin E Peterson, Mevin B Hooten, Ephraim M Hanks, and Marie-Josèe Fortin. 2018. <span>“Spatial Autoregressive Models for Statistical Inference from Ecological Data.”</span> <em>Ecological Monographs</em> 88 (1): 36–59.
</div>
<div id="ref-waldock2022" class="csl-entry" role="listitem">
Waldock, Conor, Rick D Stuart-Smith, Camille Albouy, William WL Cheung, Graham J Edgar, David Mouillot, Jerry Tjiputra, and Loı̈c Pellissier. 2022. <span>“A Quantitative Review of Abundance-Based Species Distribution Models.”</span> <em>Ecography</em> 2022 (1).
</div>
<div id="ref-willebrand2001" class="csl-entry" role="listitem">
Willebrand, Tomas, and Maria Hörnell. 2001. <span>“Understanding the Effects of Harvesting Willow Ptarmigan Lagopus Lagopus in Sweden.”</span> <em>Wildlife Biology</em> 7 (3): 205–12. <a href="https://doi.org/10.2981/wlb.2001.025">https://doi.org/10.2981/wlb.2001.025</a>.
</div>
<div id="ref-williams2002" class="csl-entry" role="listitem">
Williams, Byron K, James D Nichols, and Michael J Conroy. 2002. <em>Analysis and Management of Animal Populations</em>. Academic Press.
</div>
<div id="ref-Wilson_Martin_2011" class="csl-entry" role="listitem">
Wilson, Scott, and Kathy Martin. 2011. <span>“Life-History and Demographic Variation in an Alpine Specialist at the Latitudinal Extremes of the Range.”</span> <em>Population Ecology</em> 53 (3): 459–71. <a href="https://doi.org/10.1007/s10144-011-0261-x">https://doi.org/10.1007/s10144-011-0261-x</a>.
</div>
<div id="ref-zimmerman2010" class="csl-entry" role="listitem">
Zimmerman, G. S., W. A. Link, M. J. Conroy, J. R. Sauer, K. D. Richkus, and G. Scott Boomer. 2010. <span>“Estimating Migratory Game-Bird Productivity by Integrating Age Ratio and Banding Data.”</span> <em>Wildlife Research</em> 37 (7): 612–22. <a href="https://doi.org/10.1071/WR10062">https://doi.org/10.1071/WR10062</a>.
</div>
</div></section></div></main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      // TODO in 1.5, we should make sure this works without a callout special case
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->




</body></html>