<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.3.450">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="Chloé R. Nater, James A. Martin, Erlend B. Nilsen">

<title>Large-scale spatiotemporal variation in vital rates and population dynamics of an alpine bird</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>


<script src="Manuscript_Nater_etal_files/libs/clipboard/clipboard.min.js"></script>
<script src="Manuscript_Nater_etal_files/libs/quarto-html/quarto.js"></script>
<script src="Manuscript_Nater_etal_files/libs/quarto-html/popper.min.js"></script>
<script src="Manuscript_Nater_etal_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="Manuscript_Nater_etal_files/libs/quarto-html/anchor.min.js"></script>
<link href="Manuscript_Nater_etal_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="Manuscript_Nater_etal_files/libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="Manuscript_Nater_etal_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="Manuscript_Nater_etal_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="Manuscript_Nater_etal_files/libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">

  <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
  <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml-full.js" type="text/javascript"></script>

</head>

<body class="fullcontent">

<div id="quarto-content" class="page-columns page-rows-contents page-layout-article">

<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Large-scale spatiotemporal variation in vital rates and population dynamics of an alpine bird</h1>
</div>



<div class="quarto-title-meta">

    <div>
    <div class="quarto-title-meta-heading">Author</div>
    <div class="quarto-title-meta-contents">
             <p>Chloé R. Nater, James A. Martin, Erlend B. Nilsen </p>
          </div>
  </div>
    
  
    
  </div>
  

</header>

<section id="abstract" class="level1">
<h1>Abstract</h1>
<p>TBA</p>
</section>
<section id="introduction" class="level1">
<h1>Introduction</h1>
<!-- @JAMES & ERLEND: First draft is relatively "compact". Input on extending it and/or addition of relevant references appreciated -->
<p>There is growing demand for biodiversity indicators from international unions, national government, local management bodies, and corporate and industry actors. Such indicators ideally represent a wide range of biodiversity’s states and functions <span class="citation" data-cites="pereira2013 jetz2019">(e.g.&nbsp;Essential Biodiversity Variables, <a href="#ref-pereira2013" role="doc-biblioref">Pereira et al. 2013</a>; <a href="#ref-jetz2019" role="doc-biblioref">Jetz et al. 2019</a>)</span>, yet the development of suitable indicators for certain qualities, such as species abundance and demography, has been more difficult than for others <span class="citation" data-cites="schmeller2018 waldock2022">(<a href="#ref-schmeller2018" role="doc-biblioref">Schmeller et al. 2018</a>; <a href="#ref-waldock2022" role="doc-biblioref">Waldock et al. 2022</a>)</span>. This is at least partially due to challenging requirements regarding spatial scales of useful biodiversity indicators. On one hand, indicators need to be representative at large geographic scales, for example, in the context of countries’ reporting towards biodiversity targets <span class="citation" data-cites="feld2009">(e.g. <a href="#ref-feld2009" role="doc-biblioref">Feld et al. 2009</a>)</span>. On the other hand, indicators also ideally have good spatial resolution as the scales relevant for local-level management and planning are often much smaller <span class="citation" data-cites="stevenson2021">(<a href="#ref-stevenson2021" role="doc-biblioref">Stevenson et al. 2021</a>)</span>. This latter requirement is particularly crucial for (infrastructure) development strategies and for species management and conservation, both of which tend to require knowledge on species abundance and population dynamics (i.e.&nbsp;demographic rates) that is relevant for their county- or municipality-level decision making (+ refs?). Another reason why abundance and population indicators ideally come with good spatial resolution is that there can be substantial amounts of variation in population dynamics and life history of species across space <span class="citation" data-cites="robinson2014 horswill2019">(e.g. <a href="#ref-robinson2014" role="doc-biblioref">Robinson, Morrison, and Baillie 2014</a>; <a href="#ref-horswill2019" role="doc-biblioref">Horswill et al. 2019</a>)</span>. This is variation that needs to be accounted for in order to develop successful and sustainable strategies for area use, (harvest) management, and species and biodiversity conservation <span class="citation" data-cites="williams2002">(<a href="#ref-williams2002" role="doc-biblioref">Williams, Nichols, and Conroy 2002</a>)</span>.</p>
<p>While large-scale, spatially-explicit indicators for species abundance and populations are clearly needed, development and practical implementation are greatly limited due to the reliance of such indicators on the availability of data from large-scale, long-term monitoring programmes <span class="citation" data-cites="proencca2017">(<a href="#ref-proencca2017" role="doc-biblioref">Proença et al. 2017</a>)</span>. Consequently, many countries have been working on setting up, maintaining, and improving such monitoring programmes over the last decades. Many now well-established programmes focus on breeding birds and butterflies, and examples include the North American Breeding Bird Survey (<a href="https://www.pwrc.usgs.gov/bbs/" class="uri">https://www.pwrc.usgs.gov/bbs/</a>), UK Butterfly Monitoring Scheme (<a href="https://ukbms.org/" class="uri">https://ukbms.org/</a>), Swiss Biodiversity Monitoring (<a href="https://www.biodiversitymonitoring.ch/" class="uri">https://www.biodiversitymonitoring.ch/</a>), and many more. <!-- is there any other one we want to name here specifically? --></p>
<p>There is a natural trade-off between quality and quantity of data that can be collected in any monitoring programme: collecting high quality data in a structured manner is costly and requires trained specialists, thus limiting the amount of data that can be collected, while participatory monitoring, i.e.&nbsp;the collection of ecological data by members of the public <span class="citation" data-cites="fraisl2022">(also called citizen science, <a href="#ref-fraisl2022" role="doc-biblioref">Fraisl et al. 2022</a>)</span>, allows to greatly reduce costs and extend spatial and taxonomic scales of monitoring at the expense of data quality and risk of bias <span class="citation" data-cites="johnston2023">(<a href="#ref-johnston2023" role="doc-biblioref">Johnston, Matechou, and Dennis 2023</a>)</span>. Consequently, many large-scale monitoring programmes are often limited to presence-absence or very simple count observations, making them suitable for the development of indicators of distribution and population trends, but usually not for indicators of abundance, population dynamics, and demographic rates (+ example refs). The exception here are monitoring programmes that succeed in making use of a large number of citizen scientists that have been trained to collect data and record metadata in a structured manner and according to a carefully designed protocol. <!-- can we think of 1-2 internationally well-known examples? Check out MAPS (Monitoring of Avian Productivity and Survivorship) and CES (Constant Effort Sites) --> In Norway there is a monitoring programme for terrestrial game bird species called “Hønsefuglportalen” (= “game bird portal”, <a href="https://honsefugl.nina.no/Innsyn/en" class="uri">https://honsefugl.nina.no/Innsyn/en</a>). It is a line transect survey programme carried out annually in &gt;120 localities across the country (&gt;2000 transects) by trained volunteers using pointing dogs. The programme has a well developed protocol for recording bird observations, auxiliary data, and relevant metadata (+ref to protocol?) and established routines for quality control and annual releases of publicly available data via GBIF. As such, it is particularly well suited to become part of a workflow for producing and updating abundance and population indicators on an annual basis.</p>
<p>The line transect data from “Hønsefuglportalen” has been used previously for estimating abundance trends of willow ptarmigan (<em>Lagopus lagopus</em>) across Norway <span class="citation" data-cites="bowler2020 nilsen2020">(e.g. <a href="#ref-bowler2020" role="doc-biblioref">Diana E. Bowler et al. 2020</a>; <a href="#ref-nilsen2020" role="doc-biblioref">Nilsen and Rød-Eriksen 2020</a>)</span> but large-scale estimation of demographic rates underlying these trends has thus far remained as untapped potential of the dataset. <span class="citation" data-cites="nilsen2023">Nilsen and Nater (<a href="#ref-nilsen2023" role="doc-biblioref">2023</a>)</span> recently developed a novel integrated distance sampling model (IDSM) which successfully uses age information in line transect data alongside radio-telemetry data to estimate not just abundance, but also survival and recruitment across years. In this study, we adapt the model of <span class="citation" data-cites="nilsen2023">Nilsen and Nater (<a href="#ref-nilsen2023" role="doc-biblioref">2023</a>)</span> to run not just on a single site but on all areas with publicly available line transect data from “Hønsefuglportalen” simultaneously. Unlike several previous studies applying integrated models for population dynamics to multiple (sub-) populations separately and comparing results <span class="citation" data-cites="robinson2014 nater2023">(e.g. <a href="#ref-robinson2014" role="doc-biblioref">Robinson, Morrison, and Baillie 2014</a>; <a href="#ref-nater2023" role="doc-biblioref">Nater et al. 2023</a>)</span>, we opt for an approach explicitly integrating across space, thus allowing for sharing of information across locations and – in effect – space-for-time substitution <span class="citation" data-cites="horswill2019 morrison2022">(e.g. <a href="#ref-horswill2019" role="doc-biblioref">Horswill et al. 2019</a>; <a href="#ref-morrison2022" role="doc-biblioref">Morrison et al. 2022</a>)</span>. We then apply the resulting multi-area IDSM to “Hønsefuglportalen” data on willow ptarmigan to estimate population size, age-structure, survival, recruitment, and impacts of small rodent abundance across 41 reporting districts and 15 years (2007-2021) for this popular small-game species. We further embed the modelling workflow in a reproducible, semi-automated pipeline that will greatly facilitate the repeated calculation of abundance and population indicators at different spatial scales as new data is added every year.</p>
</section>
<section id="methods" class="level1">
<h1>Methods</h1>
<section id="study-species" class="level2">
<h2 class="anchored" data-anchor-id="study-species">Study species</h2>
<p>The willow ptarmigan (<em>Lagopus lagopus</em>) is a tetraoid bird with a circumpolar distribution, mainly inhabiting sub-alpine and arctic ecosystems <span class="citation" data-cites="fuglei2020">(see e.g. <a href="#ref-fuglei2020" role="doc-biblioref">Fuglei et al. 2020</a>)</span>. While the species is currently listed as Least Concern (LC) both in the global and Norwegian Red List of Species, it has undergone rather dramatic declines in abundance in Norway since the turn of the 20th century <span class="citation" data-cites="hjeljord2022">(<a href="#ref-hjeljord2022" role="doc-biblioref">Hjeljord and Loe 2022</a>)</span>. While the main reason for the long-term decline in abundance remain unresolved, willow ptarmigan are considered sentinel species that are sensitive to both climate change and land use changes <span class="citation" data-cites="Henden2017 Storch2007">(<a href="#ref-Henden2017" role="doc-biblioref">Henden et al. 2017</a>; <a href="#ref-Storch2007" role="doc-biblioref">Storch 2007</a>)</span>. Moreover, being one of only a handful of bird species that spend the winter in mountain ecosystems in Scandinavia, they are important components of the ecosystem as prey species for the gyrfalcon <span class="citation" data-cites="Alastair2020">(<a href="#ref-Alastair2020" role="doc-biblioref">Franke et al. 2020</a>)</span> <!-- do we have other "important" species to add here. Arctic fox? Some other endangered beasts perhaps? -->. The willow ptarmigan has a relatively fast pace of life <span class="citation" data-cites="sandercock2005 Steen1996">(<a href="#ref-sandercock2005" role="doc-biblioref">Sandercock, Martin, and Hannon 2005</a>; <a href="#ref-Steen1996" role="doc-biblioref">H. Steen and Erikstad 1996</a>)</span>, and can display substantial spatio-temporal variation in demographic rates <span class="citation" data-cites="Bowler2020">(<a href="#ref-Bowler2020" role="doc-biblioref">D. E. Bowler et al. 2020</a>)</span>. Their population dynamics are characterized by large inter-annual fluctuations in abundance <span class="citation" data-cites="hjeljord2022">(<a href="#ref-hjeljord2022" role="doc-biblioref">Hjeljord and Loe 2022</a>)</span>, and previous research has tied these fluctuations to rodent cycle through shared predators <span class="citation" data-cites="hagen1952 Bowler2020">(<a href="#ref-hagen1952" role="doc-biblioref">Hagen 1952</a>; <a href="#ref-Bowler2020" role="doc-biblioref">D. E. Bowler et al. 2020</a>)</span>. This tight relationship between the breeding success of ground nesting birds and the rodent cycle is known as the Alternative Prey Hypothesis and has been a central part of Fennoscandian research on grouse population dynamics for many decades <span class="citation" data-cites="hagen1952 Linden1988">J. B. Steen et al. (<a href="#ref-Steen1985" role="doc-biblioref">1988</a>)</span>. In addition, spring weather conditions and phenology is known to have considerable effects on breeding success and recruitment rates <span class="citation" data-cites="Eriksen2023">J. B. Steen et al. (<a href="#ref-Steen1985" role="doc-biblioref">1988</a>)</span>. Across their distributional range, they are iconic species with a high cultural value, partly linked to their popularity as game species. The latter means that information about spatio-temporal variation in demographic rates and population dynamics is particularly important in order to design sustainable harvest strategies. In addition, as a sentinel species the willow ptarmigan is well suited as an indicator species for ecosystem status; in Norway it is included in both the main national biodiversity <span class="citation" data-cites="jakobsson2020">(Nature Index for Norway, <a href="#ref-jakobsson2020" role="doc-biblioref">Jakobsson and Pedersen 2020</a>, <a href="https://www.naturindeks.no/Indicators/lirype" class="uri" role="doc-biblioref">https://www.naturindeks.no/Indicators/lirype</a>)</span> and ecosystem <span class="citation" data-cites="framstad2022">(Assessment of the Ecological Condition, <a href="#ref-framstad2022" role="doc-biblioref">Framstad et al. 2022</a>)</span> assessments.</p>
</section>
<section id="data-collection-management-and-preparation" class="level2">
<h2 class="anchored" data-anchor-id="data-collection-management-and-preparation">Data collection, management, and preparation</h2>
<section id="line-transect-sampling" class="level3">
<h3 class="anchored" data-anchor-id="line-transect-sampling">Line transect sampling</h3>
<p>The line transect survey data were collected through a structured participatory monitoring program called “Hønsefuglportalen” (<a href="https://honsefugl.nina.no/Innsyn/en" class="uri">https://honsefugl.nina.no/Innsyn/en</a>). In August each year, trained volunteer fieldworkers collect observations of willow ptarmigan and other grouse species (rock ptarmigan <em>Lagopus muta</em>, black grouse <em>Lyrurus tetrix</em>, capercaillie <em>Tetrao urogallus</em>) along predefined line transects. To increase the detection probability, each team of fieldworkers use a pointing dog to locate the birds. Each survey team typically consist of at least two persons (one dog handler and one person responsible for following the transect line) and one dog. Often, more than one dog is used for a survey, but only one dog should be used at a time. The transect lines vary in length, but are typically between 1 and 8 km (range: 0.3-16.2 km, median: 3 km). When birds are observed, the exact location of observation is noted, along with its perpendicular distance from the transect line, as well as the age and sex of the birds. An observation typically includes 1 - 12 birds (mean = 5.6), with groups &gt; 1 typically representing one brood (female and or male with young-of-the-year chicks). When the surveys are conducted in August, the chicks of the year are able to fly but can be distinguished from older birds as they are still of smaller body size. Since 2019, most of the data has been collected using a mobile app tailored made for the monitoring program, where the field workers can register and get access to the transect lines allocated to them by the local organizers. Prior to 2019, data were collected on a dedicated fieldwork form, and entered manually in a web portal afterwards. After field data has been registered, it undergoes several steps of quality control carried out by local stakeholders and personnel from the Norwegian Institute for Nature Research (NINA). Surveys are carried out on both public and private land. After an initial embargo period, all data from public land are published and made freely available as a sampling-event data set on GBIF (refs). The published datasets contain both metadata about the transect surveys (survey date, line transect length and location, study area ID, etc.) as well as bird observation data (species, number of birds of different categories (adult males, adult females, juveniles, and birds of unknown category), perpendicular distance to transect line, exact location, and time of observation). Formally, the data from public land is published as three distinct data sets, one for each of the main public land administrators (Statskog, FeFo and Fjellstyrene, respectively). <!-- I changed "owners" to "administrators", just because to me "public" implies it is not owned by anyone. But if this is wrong, please change it back --></p>
<p>Notably, the program is not designed as a centralized national monitoring programme, but rather a collection of local and regional survey programs. All involved survey areas use a common field protocol and data collection model. In addition, the local study designs are reviewed by staff at NINA, and common recommendations for study design are provided. However, due to the fact that participation by stakeholders is voluntary, the spatial distribution of transects and sampling effort is not homogeneous. In general, the effort is higher in South-Eastern and Central Norway, medium in Northern Norway, and low in Western and Southern Norway.</p>
<p>In this study we used all publicly available data for the period 2007-2021, which included a total of 2225 transects in 41 different reporting districts spanning 9 counties and 50 municipalities. Transects on which no willow ptarmigan were ever observed (i.e.&nbsp;species absence likely due to low habitat suitability) were dropped, resulting in 2077 transects used in the analyses.</p>
</section>
<section id="radio-telemetry-study-in-lierne" class="level3">
<h3 class="anchored" data-anchor-id="radio-telemetry-study-in-lierne">Radio-telemetry study in Lierne</h3>
<p>The model of <span class="citation" data-cites="nilsen2023">Nilsen and Nater (<a href="#ref-nilsen2023" role="doc-biblioref">2023</a>)</span> integrated line transect data with radio-telemetry data from from an ongoing field study of marked willow ptarmigans in Lierne municipality in Central Norway. From 2015 to 2019, around 50 birds were captured in winter (late February or early March) each year and fitted with VHF collars. The marked birds were then monitored on a regular basis until they either i) died, ii) their transmitter’s battery stopped working, or iii) we lost contact with the bird for other reasons. For most of the year, the birds were monitored <!-- how were they actually monitored? Can we have a tiny bit more detail here? --> at least once a month. During the breeding and chick-rearing season (May to July) birds were monitored more often, and during December and January we obtained few observations due to challenging field work conditions. A proportion of the birds were harvest annually in the regular recreational harvest, and birds that were harvested were reported as shot to the field personnel. In addition, as the collars had mortality switch, we were also able to locate and retrieve a high proportion of birds that died for natural causes, resulting in a known-fate mark-recapture dataset. The radio-telemetry study is described in detail in <span class="citation" data-cites="Israelsen2020">Israelsen et al. (<a href="#ref-Israelsen2020" role="doc-biblioref">2020</a>)</span> and in <span class="citation" data-cites="Arnekleiv2022">Arnekleiv et al. (<a href="#ref-Arnekleiv2022" role="doc-biblioref">2022</a>)</span>.</p>
<p>In this study we used data from years 2015 - 2020, and the total sample size across these years was 139 birds for the Aug-Jan period and 258 birds for the Feb-Jul period.</p>
</section>
</section>
<section id="national-scale-integrated-model" class="level2">
<h2 class="anchored" data-anchor-id="national-scale-integrated-model">National-scale integrated model</h2>
<section id="integrated-distance-sampling-model-idsm-for-willow-ptarmigan" class="level3">
<h3 class="anchored" data-anchor-id="integrated-distance-sampling-model-idsm-for-willow-ptarmigan">Integrated distance sampling model (IDSM) for willow ptarmigan</h3>
<p><span class="citation" data-cites="nilsen2023">Nilsen and Nater (<a href="#ref-nilsen2023" role="doc-biblioref">2023</a>)</span> recently developed an integrated distance sampling model (IDSM) which jointly analyses line transect and radio-telemetry data and applied it to willow ptarmigan in the Western part of Lierne municipality in Norway. The model consists of a deterministic population model with two age classes (juveniles and adults) and four data likelihoods: 1) likelihood for observation distances from transect lines for estimating detection probability; 2) likelihood for age-specific counts on transect surveys for estimating numbers of juveniles and adults present; 3) likelihood for juvenile to adult ratios observed at the locality level to provide estimate recruitment rate (as juveniles/adult); and 4) likelihood for known-fate telemetry data to estimate seasonal and annual survival. Below, we describe our new extension of this model to include data from several as opposed to just one. For more detailed information on the single-site model, including tests of model performance, see <span class="citation" data-cites="nilsen2023">Nilsen and Nater (<a href="#ref-nilsen2023" role="doc-biblioref">2023</a>)</span>.</p>
</section>
<section id="multi-area-model-extension" class="level3">
<h3 class="anchored" data-anchor-id="multi-area-model-extension">Multi-area model extension</h3>
<p>For applying the ptarmigan IDSM across all 41 reporting districts we included an area index in all model parameters (<a href="#fig-model">Figure&nbsp;1</a>) and enabled sharing of information among areas by explicitly modelling spatial variation alongside shared temporal and residual variation in vital rates and detection parameters.</p>
<p>The spatially-explicit formulation of the two age-class population model can be written as:</p>
<p><span class="math display">\[
\begin{aligned}
  D_{juv, x, j, t+1} &amp; = D_{ad, x, j, t+1} * R_{x, t+1} \\
  D_{ad, x, j, t+1} &amp; = S_{x, t} * (D_{juv, x, j, t} + D_{ad, x, j, t})  
\end{aligned}
\]</span></p>
<p>Here, <span class="math inline">\(D_{juv, j, x, t}\)</span> and <span class="math inline">\(D_{ad, j, x, t}\)</span> are the densities of juvenile and adult ptarmigan in survey site (= transect) <span class="math inline">\(j\)</span> of area <span class="math inline">\(x\)</span> in year <span class="math inline">\(t\)</span>, respectively. Both juveniles and adult survive from year <span class="math inline">\(t\)</span> to <span class="math inline">\(t+1\)</span> with an area- (<span class="math inline">\(x\)</span>) and year- (<span class="math inline">\(t\)</span>) specific survival probability <span class="math inline">\(S_{x, t}\)</span>, and survivors produce the next generation of juveniles according to an area- and year-specific recruitment rate (<span class="math inline">\(R_{x,t}\)</span>).</p>
<div id="fig-model" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="Figures/RypeIDSM_multiArea_structure.png" class="img-fluid figure-img"></p>
<figcaption class="figure-caption">Figure&nbsp;1: Simplified graphical representation of the ptarmigan life cycle with two age classes and the data sources included in the integrated distance sampling model. The pink “x”’s indicate the added dimension for area. Juv[x,t] = juveniles in year t. Ad[x,t] = adults in year t. R[x,t] = recruitment rate in year t. S[x,t] = survival probability from year t to t+1. Note that the additional site (=transect) dimension, “j”, is omitted for the sake of illustration.</figcaption>
</figure>
</div>
<p>The initial densities, <span class="math inline">\(D_{juv, x, j, 1}\)</span> and <span class="math inline">\(D_{ad, x, j, 1}\)</span> are modelled for each site (= transect) as random realizations of log normal distributions with area-specific log means (<span class="math inline">\(\mu^{D1}_x\)</span>) and log standard deviations (<span class="math inline">\(\sigma^{D1}_x\)</span>). Survival (<span class="math inline">\(S_{x,t}\)</span>) and recruitment (<span class="math inline">\(R_{x,t}\)</span>), on the other hand, are assumed to be the same for all sites <span class="math inline">\(j\)</span> within a given area <span class="math inline">\(x\)</span> and were modelled as:</p>
<p><span class="math display">\[
\begin{aligned}
  logit(S_{x, t}) &amp; = logit(\mu^S) + \varepsilon^{X.S}_{x} + \varepsilon^{T.S}_{t} + \varepsilon^{R.S}_{x,t} \\
    log(R_{x, t}) &amp; = log(\mu^R) + \beta_x*rodentOcc_{x,t} +  \varepsilon^{X.R}_{x} + \varepsilon^{T.R}_{t} + \varepsilon^{R.R}_{x,t}
\end{aligned}
\]</span> The global means ,<span class="math inline">\(\mu\)</span>, and the normally distributed spatial random effects, <span class="math inline">\(\varepsilon^X\)</span>, represent the equivalent of what is elsewhere referred to as “hyper-parameter distributions” for sharing information on demographic rates across areas <span class="citation" data-cites="horswill2019 horswill2021">(e.g. <a href="#ref-horswill2019" role="doc-biblioref">Horswill et al. 2019</a>, <a href="#ref-horswill2021" role="doc-biblioref">2021</a>)</span>. We also used this same approach for defining the area-specific effects (<span class="math inline">\(\beta_x\)</span>) of local yearly rodent occupancy (<span class="math inline">\(rodentOcc_{x,t}\)</span>) on recruitment. In addition to spatial variation in survival and recruitment, we also included large-scale temporal variation through random year effects that were shared by across all areas (<span class="math inline">\(\varepsilon^T_t\)</span>) and otherwise unaccounted for variation through year- and area-specific residual random effects (<span class="math inline">\(\varepsilon^R_{x,t}\)</span>). Spatial, temporal, and residual random effects were modelled as normally distributed with globally defined (= shared) standard deviations.</p>
<p>The three likelihoods for data resulting from the line transect sampling (observation distances, age-specific counts, and juvenile to adult ratios, see above) were also formulated as spatially explicit, with year- and area-specific distance sampling detection parameters modelled in the same way as survival and recruitment (bar the effect of rodent occupancy, <a href="#fig-model">Figure&nbsp;1</a>). For the known-fate telemetry data (and the seasonal decomposition of survival estimated from it), on the other hand, we did not add an additional area dimension as this data was only available for the Lierne area.</p>
</section>
<section id="model-implementation" class="level3">
<h3 class="anchored" data-anchor-id="model-implementation">Model implementation</h3>
<p>We implemented our multi-area IDSM in a Bayesian framework using NIMBLE version 1.0.1 <span class="citation" data-cites="valpine2017">(<a href="#ref-valpine2017" role="doc-biblioref">Valpine et al. 2017</a>)</span> in R version 4.3.1 <span class="citation" data-cites="R2023">(<a href="#ref-R2023" role="doc-biblioref"><strong>R2023?</strong></a>)</span>. For the likelihood for line transect observation distances we used a custom half-normal distribution developed by Michael Scroggie in the “nimbleDistance” package (https://github.com/scrogster/nimbleDistance). We used non-informative uniform priors for all parameters and including biologically sensible boundaries where possible. We simulated complete sets of initial values for all model nodes prior to model running and using pre-defined seeds to ensure reproducibility. Using NIMBLE’s standard samplers, we then ran 4 MCMC chains of 100k iterations each. We discarded the first 40k samples of each chain as burn-in, and thinned the remainder by a factor 20, resulting in a final joint posterior containing a total of 4 x 3k = 120k samples (note that high thinning rates were necessary to constrain memory load of the joint posterior, which included 314568 monitored parameters).</p>
</section>
</section>
<section id="post-hoc-variance-decomposition" class="level2">
<h2 class="anchored" data-anchor-id="post-hoc-variance-decomposition">Post-hoc variance decomposition</h2>
<p>Following model fitting, we calculated posterior distributions for the proportions of variance in survival probabilities, recruitment rates, and detection decay explained by 1) spatial variation (<span class="math inline">\(var_{area}\)</span>), 2) temporal variation (<span class="math inline">\(var_{year}\)</span>), 3) residual variation (<span class="math inline">\(var_{residual}\)</span>), and 4) variation in rodent occupancy (<span class="math inline">\(var_{rodent}\)</span>). To obtain the proportion variance explained by each of the component, we divided it by the sum of all other components (<span class="math inline">\(var_{area} + var_{year} + var_{residual} + var_{rodent}\)</span>). The spatial, temporal, and residual variance components were defined as the square of the estimated corresponding random effects standard deviation from the model while <span class="math inline">\(var_{rodent}\)</span> was calculated as the variance of all area- and year-specific <span class="math inline">\(\beta_x*rodentOcc_{x,t}\)</span> products.</p>
</section>
<section id="reproducible-workflow-with-targets" class="level2">
<h2 class="anchored" data-anchor-id="reproducible-workflow-with-targets">Reproducible workflow with “targets”</h2>
<p>Reproducibility and ease of repeating analyses was a key focus when developing the multi-area IDSM.To that end, we set up the workflow as a “targets pipeline”, implemented through the R package “targets” <span class="citation" data-cites="landau2021">(<a href="#ref-landau2021" role="doc-biblioref">Landau 2021</a>)</span>. The pipeline contains a variety of options for controlling modelling decisions in the workflow such as the year range of data to consider, the level of spatial aggregation (i.e.&nbsp;reporting district vs.&nbsp;survey locality), whether to model time variation in survival and/or effects of rodent abundance, whether to run MCMC chains sequentially or in parallel, etc. A visual representation of the pipeline is also displayed in <a href="#fig-workflow">Figure&nbsp;2</a> and for more details on pipeline implementation and options, we refer the reader to the GitHub repository: <a href="https://github.com/ErlendNilsen/OpenPop_Integrated_DistSamp" class="uri">https://github.com/ErlendNilsen/OpenPop_Integrated_DistSamp</a>.</p>
<div id="fig-workflow" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="Figures/Workflow_layout.png" class="img-fluid figure-img" width="424"></p>
<figcaption class="figure-caption">Figure&nbsp;2: Graphical representation of the “targets pipeline” for the multi-area modelling setup. Upward facing triangles are functions, downward facing triangles are general options/arguments, circles are objects and outputs (=“targets”) created as part of the workflow.</figcaption>
</figure>
</div>
</section>
</section>
<section id="results" class="level1">
<h1>Results</h1>
<p>All numerical results in the following are presented as median [95% credible interval] unless otherwise indicated. Posterior summaries (median, 95% credible interval, mean, standard deviation, coefficient of variation) for all main parameters are also provided in the supplementary file “PosteriorSummaries_byAreas.csv”. Supplementary figures (SF) are provided as .pdf files with captions in “SuppFigures_Captions.txt”.</p>
<section id="population-density" class="level2">
<h2 class="anchored" data-anchor-id="population-density">Population density</h2>
<p>Only during the most recent four years (2018-2021) has data been collected regularly for all reporting areas included in the analyses. During this period, estimated population densities varied between 2.2 [1.547, 3.067] birds/km^2 in the area “Statskog og Klinga utm.” close to the coast in central Norway to 55.906 [51.679, 60.093] birds/km^2 in “Ålen og Haltdalen Fjellstyre” further south near the Swedish border. In general, population density appeared to be lowest in the Northern parts of Norway and highest in the eastern part of central Norway (<a href="#fig-densA">Figure&nbsp;3 (a)</a>). Uncertainty in density estimates was relatively consistent, with a few areas (inlcuding the one with the lowest estimated density, “Statskog og Klinga utm.”) sticking out by having substantially less precise estimates (<a href="#fig-densB">Figure&nbsp;3 (b)</a>). Populations fluctuated substantially over time in any given area (SF “TimeSeries_popDens1.pdf”) and some years seemed to be indicative of relatively high (e.g.&nbsp;2011, 2014, 2018) or low (e.g.&nbsp;2012, 2015) densities across a large number of areas.</p>
<div id="fig-dens" class="quarto-layout-panel">
<figure class="figure">
<div class="quarto-layout-row quarto-layout-valign-top">
<div class="quarto-layout-cell quarto-layout-cell-subref" style="flex-basis: 50.0%;justify-content: center;">
<div id="fig-densA" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="Figures/Avg_popDens_Map-4.png" class="img-fluid figure-img" data-ref-parent="fig-dens"></p>
<figcaption class="figure-caption">(a) Median density estimates</figcaption>
</figure>
</div>
</div>
<div class="quarto-layout-cell quarto-layout-cell-subref" style="flex-basis: 50.0%;justify-content: center;">
<div id="fig-densB" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="Figures/Avg_popDens_Map-6.png" class="img-fluid figure-img" data-ref-parent="fig-dens"></p>
<figcaption class="figure-caption">(b) Uncertainty in density estimates</figcaption>
</figure>
</div>
</div>
</div>
<p></p><figcaption class="figure-caption">Figure&nbsp;3: Median (a) and coefficient of variation (standard deviation / mean) (b) of posterior estimates of average ptarmigan density in the four most recent years (2018-2021) across 41 reporting areas (summarised at the municipality level) in Norway. Darker colors indicate higher median values and higher uncertainty.</figcaption><p></p>
</figure>
</div>
</section>
<section id="population-growth-rate" class="level2">
<h2 class="anchored" data-anchor-id="population-growth-rate">Population growth rate</h2>
<p>Average population growth rates over the last four years (2018-2021) ranged from moderate declines (0.72 [0.643, 0.919] in the “Kongsvoll” area) to &gt; 50% increase (1.565 [1.268, 1.973] in the “Statskog og Klinga utm.” area). In the majority of reporting areas (24 out of 41), populations of willow ptarmigan have been increasing over the period 2018-2021 (<a href="#fig-lam">Figure&nbsp;4</a>). Some areas – predominantly in central Norway – also had declining populations, but many of those declines followed upon periods of increase between the start of data collection in 2007 and sometime between 2016 and 2018 (SF “TimeSeries_popDens1.pdf”).</p>
<div id="fig-lam" class="quarto-layout-panel">
<figure class="figure">
<div class="quarto-layout-row quarto-layout-valign-top">
<div class="quarto-layout-cell quarto-layout-cell-subref" style="flex-basis: 50.0%;justify-content: center;">
<div id="fig-lamA" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="Figures/Avg_lambda_Map-4.png" class="img-fluid figure-img" data-ref-parent="fig-lam"></p>
<figcaption class="figure-caption">(a) Median population growth rate estimates</figcaption>
</figure>
</div>
</div>
<div class="quarto-layout-cell quarto-layout-cell-subref" style="flex-basis: 50.0%;justify-content: center;">
<div id="fig-lamB" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="Figures/Avg_lambda_Map-5.png" class="img-fluid figure-img" data-ref-parent="fig-lam"></p>
<figcaption class="figure-caption">(b) Uncertainty in population growth rate estimates</figcaption>
</figure>
</div>
</div>
</div>
<p></p><figcaption class="figure-caption">Figure&nbsp;4: Median (a) and standard deviation (b) of posterior estimates of average annual population growth rate over the four most recent years (2018-2021) across 41 reporting areas (summarised at the municipality level) in Norway. In a), pinkish colors indicate declining populations while greenish colors indicate growing populations (white = stable populations). In b) darker colors indicate higher uncertainty.</figcaption><p></p>
</figure>
</div>
<p>The highest population growth rates were estimated for areas with relatively low population densities across latitudes <a href="#fig-paramCorrs">Figure&nbsp;5</a> but we did not find evidence for a strong association between population growth rates and population densities overall.</p>
<div id="fig-paramCorrs" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="Figures/AllParams_Latitude.png" class="img-fluid figure-img"></p>
<figcaption class="figure-caption">Figure&nbsp;5: Posterior summaries (points = posterior medians, lines = 95% credible intervals) of area-specific population growth rate vs.&nbsp;population density (A), recruitment rate vs.&nbsp;survival probability (B) and rodent effect on recruitment along a longitudinal gradient (C). Color indicates latitude of the midpoint of each area.</figcaption>
</figure>
</div>
</section>
<section id="survival-probabilities-and-recruitment-rates" class="level2">
<h2 class="anchored" data-anchor-id="survival-probabilities-and-recruitment-rates">Survival probabilities and recruitment rates</h2>
<p>Annual survival probabilities ranged from 0.373 [0.282, 0.417] (area “Statskog og Klinga utm.”) to 0.444 [0.385, 0.519] (area “Øst Finnmark”) across reporting areas in Norway, with the highest values occurring in the far north and in the mountains in the south (<a href="#fig-pVRA">Figure&nbsp;6 (a)</a>). The global average survival probability across all areas and years (<span class="math inline">\(\mu^S\)</span>) was estimated at None, 0.405 [0.382, 0.422]. Spatial variation in survival (random effect SD = 0.125 [0.003, 0.252]) was relatively low compared to temporal (0.587 [0.393, 0.947]) and residual (0.641 [0.584, 0.706]) variation.</p>
<p>Recruitment rates varied between 1.738 [1.351, 2.322] (area “Gausdal Fjellstyre”) and 3.449 [2.615, 4.792] (area “Statskog og Klinga utm.”) and displayed a spatial pattern opposite to that of annual survival, i.e.&nbsp;lower recruitment rates co-occurring with higher survival rates and vice-versa [<a href="#fig-pVRB">Figure&nbsp;6 (b)</a>; <a href="#fig-paramCorrs">Figure&nbsp;5</a> B). Across all areas and years, average recruitment rate was 2.56 [2.286, 3.316]. Unlike for survival, the model predicted similar magnitudes of spatial and temporal variation (random effect SDs of 0.172 [0.124, 0.312] and 0.151 [0.079, 0.364], respectively), and about twice as much residual variation (0.328 [0.305, 0.354]).</p>
<div id="fig-pVR" class="quarto-layout-panel">
<figure class="figure">
<div class="quarto-layout-row quarto-layout-valign-top">
<div class="quarto-layout-cell quarto-layout-cell-subref" style="flex-basis: 50.0%;justify-content: center;">
<div id="fig-pVRA" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="Figures/Avg_pSurv_Map-1.png" class="img-fluid figure-img" data-ref-parent="fig-pVR"></p>
<figcaption class="figure-caption">(a) Median survival probabilities</figcaption>
</figure>
</div>
</div>
<div class="quarto-layout-cell quarto-layout-cell-subref" style="flex-basis: 50.0%;justify-content: center;">
<div id="fig-pVRB" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="Figures/Avg_rRep_Map-1.png" class="img-fluid figure-img" data-ref-parent="fig-pVR"></p>
<figcaption class="figure-caption">(b) Median recruitment rates</figcaption>
</figure>
</div>
</div>
</div>
<p></p><figcaption class="figure-caption">Figure&nbsp;6: Posterior medians of average annual survival probabilities (a) and recruitment rates (b) across 41 reporting areas (summarised at the municipality level) in Norway. Darker colors indicate higher median values. Measures for corresponding uncertainty in estimates are visualized in SFs “Avg_pSurv_Map.pdf” and “Avg_rRep_Map.pdf” for survival and recruitment, respectively.</figcaption><p></p>
</figure>
</div>
<p>The MCMC chains for many of the area-specific average survival probabilities and recruitment rates, as well as for the global averages (intercepts) for both vital rates, were mixing rather poorly. Despite that, mixing was good and resulting posteriors well defined for the area- and year-specific estimates of survival and recruitment (SF “PostDens_tS_tR.pdf”). There was substantial variation in both vital rates across time (SFs “TimeSeries_pSurv.pdf” and “TimeSeries_rRep.pdf”). In many areas, the years 2011, 2014, and 2018 not only sported high population densities (see above) but were also characterized by both high recruitment and low subsequent survival. The overall low density years 2012 and 2015, conversely, often featured lower recruitment and, in some cases, higher survival. Notably, there were also years with very little spatial synchrony, i.e.&nbsp;very different relative yearly survival probabilities and recruitment rates (e.g.&nbsp;2010 and 2020 for survival and 2013, 2016, and 2017 for recruitment).</p>
</section>
<section id="effects-of-rodent-occupancy" class="level2">
<h2 class="anchored" data-anchor-id="effects-of-rodent-occupancy">Effects of rodent occupancy</h2>
<p>The model predicted a positive global effect of rodent occupancy on recruitment rate (average slope on the log scale = 0.073 [0.033, 0.121]). Nonetheless, spatial variation in the rodent effect was substantial (random effect SD = 0.088 [0.032, 0.14]). This resulted in negative median effects in 5 areas, positive median effects in 36 areas, and a range of effect sizes from -0.034 [-0.182, 0.097] (area “Selbu Fjellstyre”) to 0.16 [0.036, 0.298] (area “Gausdal Fjellstyre”, <a href="#fig-paramCorrs">Figure&nbsp;5</a> C). The largest positive rodent effects were estimated for areas in the very North of Norway, as well as in the mountaineous regions in the central and southwestern parts of the country (SF “betaR_Map.pdf”). Effects with negative posterior medians were located mostly at intermediate latitudes, but we note that all of these had posterior distributions featuring substantial overlap with 0 (<a href="#fig-paramCorrs">Figure&nbsp;5</a> C).</p>
</section>
<section id="detection-parameters" class="level2">
<h2 class="anchored" data-anchor-id="detection-parameters">Detection parameters</h2>
<p>TBA</p>
</section>
<section id="variance-decomposition" class="level2">
<h2 class="anchored" data-anchor-id="variance-decomposition">Variance decomposition</h2>
<p>The relative importance of different components for explaining parameter variation differed among recruitment rate, survival probability, and detection decay (<a href="#fig-varDecomp">Figure&nbsp;7</a>). The largest portion of variation in recruitment was attributed to residual variation (60.5 [36.4, 74.7] %), followed by spatial (16.4 [8.1, 40.6] %) and temporal (12.6 [4, 47] %) variation. Rodent occupancy, which contains both a spatial and a temporal dimension, explained 6.6 [2.7, 12.3] % of the total variation. For survival, there was large uncertainty in the estimated proportions of variance explained by different components. The model predicted similar potential contributions from temporal (44.5 [26.2, 67.9] %) and residual (53 [30.3, 72.6] %) variation and suggested that spatial variation was only responsible for 1.8 [0, 7.8] % of the total variance. Total variance in detection decay was attributed evenly to spatial, temporal, and residual variation at 33 [18.5, 49.4] %, 33.4 [17.8, 57.7] %, and 32 [19.6, 45] %, respectively.</p>
<div id="fig-varDecomp" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="Figures/VarDecomposition_overall.png" class="img-fluid figure-img"></p>
<figcaption class="figure-caption">Figure&nbsp;7: Posterior distributions for the proportions of parameter variance explained by spatial (blue), temporal (orange), and residual (green) variation, as well as by effects of rodent occupancy (yellow).</figcaption>
</figure>
</div>
</section>
</section>
<section id="discussion" class="level1">
<h1>Discussion</h1>
<p>TBA</p>
<section id="abundance-and-vital-rates-across-space-and-time" class="level2">
<h2 class="anchored" data-anchor-id="abundance-and-vital-rates-across-space-and-time">Abundance and vital rates across space and time</h2>
<ul>
<li>Biological results (interpretations and comparisons with other case studies / species / systems)</li>
<li>With some focus on what we are able to get insights on thanks to the integration across time and space (i.e.&nbsp;better estimates through data sharing, relative importance of different kinds of variation, etc. ) <!-- @ERLEND & @JAMES, once the "new" results are in: thoughts on what we see that is interesting, cool, expected, unexpected etc. Can we compare to other large scale studies, either on other grouse-like creatures or harvested species more generally? --></li>
</ul>
<p>Annual recruitment rates showed a spatial pattern that was inverse relative to that of annual survival, with lower recruitment rates co-occurring with higher survival rates and vice-versa. This is consistent with basic life history theory <span class="citation" data-cites="stearns1992">(<a href="#ref-stearns1992" role="doc-biblioref">Stearns 1992</a>)</span> and with previous studies finding relatively slower life histories in alpine / high altitude areas <span class="citation" data-cites="sandercock2005 bears2009 alice2016">(e.g. <a href="#ref-sandercock2005" role="doc-biblioref">Sandercock, Martin, and Hannon 2005</a>; <a href="#ref-bears2009" role="doc-biblioref">Bears, Martin, and White 2009</a>; <a href="#ref-alice2016" role="doc-biblioref">Alice Boyle, Sandercock, and Martin 2016</a>)</span>.</p>
</section>
<section id="implications-for-management" class="level2">
<h2 class="anchored" data-anchor-id="implications-for-management">Implications for management</h2>
<!-- preliminary section title) -->
<ul>
<li>What results mean for management (of habitat and of harvest)</li>
</ul>
</section>
<section id="model-limitations-and-outlook" class="level2">
<h2 class="anchored" data-anchor-id="model-limitations-and-outlook">Model limitations and outlook</h2>
<ul>
<li><p>Some limitations of “base model” that need mentioning (e.g.&nbsp;age assignment, ignoring potential effects of flock size on detection)</p></li>
<li><p>Limited usefulness of auxiliary radio-telemetry data as only from one area (big potential for including additional datasets from other small-scale studies, telemetry but also other, e.g.&nbsp;nest monitoring?) <!-- @ERLEND: Any thoughts on what the status here is? Might there be other data available from beyond Lierne? Are there exciting future prospects e.g. linked to automated monitoring with cameras at a larger scale? --></p></li>
<li><p>Outlook: improved modelling of spatio-temporal patterns through correlated REs</p></li>
</ul>
<p>Another approach to maximising information gain when jointly analysing data from different areas is explicit modelling of spatial and spatio-temporal variation. While this is commonly done for modern species distribution models <span class="citation" data-cites="pacifici2017 guelat2018">(e.g. <a href="#ref-pacifici2017" role="doc-biblioref">Pacifici et al. 2017</a>; <a href="#ref-guelat2018" role="doc-biblioref">Guélat and Kéry 2018</a>)</span>, it has rarely been implemented in demographic models. This is not least due to the fact that few demographic models span large spatial extents. Since our IDSM and the willow ptarmigan data do actually do that though, extending the framework to account for spatial and spatiotemporal correlations may be feasible. Per now, we have not yet attempted this, but are planning to do so in the near future. (BUT: we do already have some computational limitations now, so likely to need development of more efficient sampling strategies. Which is fortunately possible with NIMBLE but there is a chance that another platform, e.g.&nbsp;INLA, may be required) <!-- @ERLEND: related to this, I want to discuss the potential of including the not publicly available data. Is there an overview somewhere over what the coverage (in terms of municipalities) would be if we had access to all the Hønsefuglportalen data? This might matter not just for increasing sample size, but perhaps also making CAR modelling (spatial structure through neighborhood relations) possible --></p>
<!-- @MYSELF: Have a closer look at chapter 19 in IPM book to see what they did there and also what other work they reference -->
<ul>
<li>Outlook: integration with / coupling to models for harvest to develop tools for (automatic) calculation of e.g.&nbsp;hunting quotas <!-- @JAMES: This one is for you. Some ruminations on how and why it would be great to formally model harvest, integration with hunting data, maybe eventually develop a tool to calculate quotas (I know this has been developed for other game species), etc. --></li>
</ul>
</section>
<section id="reproducible-workflows-for-a-sustainable-future" class="level2">
<h2 class="anchored" data-anchor-id="reproducible-workflows-for-a-sustainable-future">Reproducible workflows for a sustainable future</h2>
<ul>
<li>Structure / setup of this workflow (from data collection to analytical results)</li>
<li>Why this matters: transparency and credibility, feedback to volunteer data collectors, cost-effective to re-do analyses for annual/regular reporting, easy to re-use and build upon, publicly accessible and well documented to allow more effective resource use in follow-up projects (sustainable use of funding), etc.</li>
<li>Also: useful to link up to different reporting routes and/or more general (multi-indicator) projects <!-- @EVERYONE: Do we know of some good references for this? PRseumably we cannot cite our chapter in the Technical Manual yet :-P --></li>
</ul>
</section>
<section id="conclusion" class="level2">
<h2 class="anchored" data-anchor-id="conclusion">Conclusion</h2>
<ul>
<li>Summarise neatly</li>
<li>Things worth highlighting: inference across large spatial scale but with “useful” resolution for management, advantages of integration across space (we can estimate better and more), integration of small locality-specific datasets with large national-scale monitoring, importance of reproducible workflows</li>
</ul>
</section>
</section>
<section id="acknowledgements" class="level1">
<h1>Acknowledgements</h1>
<p>TBA … Additionally, we would like to thank Bernardo Brandão Niebuhr dos Santos for his help with spatial data visualization. <!-- @ERLEND: you maybe know best all the people/institutions we need to thank here --> <!-- @JAMES: should we be acknowledging your fullbright scholarship which made the collab possible? If so, please add details. --></p>
</section>
<section id="data-and-code-availability" class="level1">
<h1>Data and code availability</h1>
<p>The raw data from the line transect surveys is deposited on GBIF and can be accessed freely via the Living Norway Data Portal (https://data.livingnorway.no/). The work presented above is based on versions 1.7, 1.8, and 1.12 for the datasets from Fjellstyrene (key b49a2978-0e30-4748-a99f-9301d17ae119), Statskog (key 6a948a1c-7e23-4d99-b1c1-ec578d0d3159), and FeFo (key c47f13c1-7427-45a0-9f12-237aad351040), respectively.</p>
<p>The auxiliary radio-telemetry data, rodent occupancy data, and all code including the “targets pipeline” can be found in the project’s repository on GitHub: <a href="https://github.com/ErlendNilsen/OpenPop_Integrated_DistSamp" class="uri">https://github.com/ErlendNilsen/OpenPop_Integrated_DistSamp</a>. The results presented in this paper were created using version 2.0 of the code (Zenodo citation TBA).</p>
</section>
<section id="references" class="level1 unnumbered">


</section>

<div id="quarto-appendix" class="default"><section class="quarto-appendix-contents" role="doc-bibliography"><h2 class="anchored quarto-appendix-heading">References</h2><div id="refs" class="references csl-bib-body hanging-indent" role="list">
<div id="ref-alice2016" class="csl-entry" role="listitem">
Alice Boyle, W, Brett K Sandercock, and Kathy Martin. 2016. <span>“Patterns and Drivers of Intraspecific Variation in Avian Life History Along Elevational Gradients: A Meta-Analysis.”</span> <em>Journal of Animal Ecology</em> 91 (2): 469–82.
</div>
<div id="ref-Arnekleiv2022" class="csl-entry" role="listitem">
Arnekleiv, Øyvind, Katrine Eldegard, Pål F. Moa, Lasse F. Eriksen, and Erlend B. Nilsen. 2022. <span>“Drivers and Consequences of Partial Migration in an Alpine Bird Species.”</span> Journal Article. <em>Ecology and Evolution</em> 12 (3): e8690. https://doi.org/<a href="https://doi.org/10.1002/ece3.8690">https://doi.org/10.1002/ece3.8690</a>.
</div>
<div id="ref-bears2009" class="csl-entry" role="listitem">
Bears, H, K Martin, and GC White. 2009. <span>“Breeding in High-Elevation Habitat Results in Shift to Slower Life-History Strategy Within a Single Species.”</span> <em>Journal of Animal Ecology</em> 78 (2): 365–75.
</div>
<div id="ref-Bowler2020" class="csl-entry" role="listitem">
Bowler, D. E., M. A. J. Kvasnes, H. C. Pedersen, B. K. Sandercock, and E. B. Nilsen. 2020. <span>“Impacts of Predator-Mediated Interactions Along a Climatic Gradient on the Population Dynamics of an Alpine Bird.”</span> <em>Proc Biol Sci</em> 287 (1941): 20202653. <a href="https://doi.org/10.1098/rspb.2020.2653">https://doi.org/10.1098/rspb.2020.2653</a>.
</div>
<div id="ref-bowler2020" class="csl-entry" role="listitem">
Bowler, Diana E, Mikkel AJ Kvasnes, Hans C Pedersen, Brett K Sandercock, and Erlend B Nilsen. 2020. <span>“Impacts of Predator-Mediated Interactions Along a Climatic Gradient on the Population Dynamics of an Alpine Bird.”</span> <em>Proceedings of the Royal Society B</em> 287 (1941): 20202653.
</div>
<div id="ref-Eriksen2023" class="csl-entry" role="listitem">
Eriksen, Lasse Frost, Thor Harald Ringsby, Hans Chr. Pedersen, and Erlend B. Nilsen. 2023. <span>“Climatic Forcing and Individual Heterogeneity in a Resident Mountain Bird: Legacy Data Reveal Effects on Reproductive Strategies.”</span> Journal Article. <em>Royal Society Open Science</em> 10 (5): 221427. <a href="https://doi.org/doi:10.1098/rsos.221427">https://doi.org/doi:10.1098/rsos.221427</a>.
</div>
<div id="ref-feld2009" class="csl-entry" role="listitem">
Feld, Christian K, Pedro Martins da Silva, José Paulo Sousa, Francesco De Bello, Rob Bugter, Ulf Grandin, Daniel Hering, et al. 2009. <span>“Indicators of Biodiversity and Ecosystem Services: A Synthesis Across Ecosystems and Spatial Scales.”</span> <em>Oikos</em> 118 (12): 1862–71.
</div>
<div id="ref-fraisl2022" class="csl-entry" role="listitem">
Fraisl, Dilek, Gerid Hager, Baptiste Bedessem, Margaret Gold, Pen-Yuan Hsing, Finn Danielsen, Colleen B Hitchcock, et al. 2022. <span>“Citizen Science in Environmental and Ecological Sciences.”</span> <em>Nature Reviews Methods Primers</em> 2 (1): 64.
</div>
<div id="ref-framstad2022" class="csl-entry" role="listitem">
Framstad, Erik, Nina E Eide, Wenche Eide, Kari Klanderud, Anders Kolstad, Joachim Töpper, and Vigdis Vandvik. 2022. <span>“Vurdering Av <span class="nocase">ø</span>kologisk Tilstand for Fjell i Norge i 2021.”</span> <em>NINA Rapport</em> 2050.
</div>
<div id="ref-Alastair2020" class="csl-entry" role="listitem">
Franke, Alastair, Knud Falk, Kevin Hawkshaw, Skip Ambrose, David L. Anderson, Peter J. Bente, Travis Booms, et al. 2020. <span>“Status and Trends of Circumpolar Peregrine Falcon and Gyrfalcon Populations.”</span> Journal Article. <em>Ambio</em> 49 (3): 762–83. <a href="https://doi.org/10.1007/s13280-019-01300-z">https://doi.org/10.1007/s13280-019-01300-z</a>.
</div>
<div id="ref-fuglei2020" class="csl-entry" role="listitem">
Fuglei, Eva, John-André Henden, Chris T. Callahan, Olivier Gilg, Jannik Hansen, Rolf A. Ims, Arkady P. Isaev, et al. 2020. <span>“Circumpolar Status of Arctic Ptarmigan: Population Dynamics and Trends.”</span> <em>Ambio</em> 49 (3): 749–61. <a href="https://doi.org/10.1007/s13280-019-01191-0">https://doi.org/10.1007/s13280-019-01191-0</a>.
</div>
<div id="ref-guelat2018" class="csl-entry" role="listitem">
Guélat, Jérôme, and Marc Kéry. 2018. <span>“Effects of Spatial Autocorrelation and Imperfect Detection on Species Distribution Models.”</span> <em>Methods in Ecology and Evolution</em> 9 (6): 1614–25.
</div>
<div id="ref-hagen1952" class="csl-entry" role="listitem">
Hagen, Yngvar. 1952. <em>Analysis and Management of Animal Populations</em>. Gyldendal Norsk forlag.
</div>
<div id="ref-Henden2017" class="csl-entry" role="listitem">
Henden, John-André, Rolf Anker Ims, Eva Fuglei, and Åshild Ønvik Pedersen. 2017. <span>“Changed Arctic-Alpine Food Web Interactions Under Rapid Climate Warming: Implication for Ptarmigan Research”</span> 2017 (SP1): wlb.00240. https://doi.org/<a href="https://doi.org/10.2981/wlb.00240">https://doi.org/10.2981/wlb.00240</a>.
</div>
<div id="ref-hjeljord2022" class="csl-entry" role="listitem">
Hjeljord, Olav, and Leif Egil Loe. 2022. <span>“The Roles of Climate and Alternative Prey in Explaining 142 Years of Declining Willow Ptarmigan Hunting Yield.”</span> <em>Wildlie Biology</em> 2022 (6): e01058. https://doi.org/<a href="https://doi.org/10.1002/wlb3.01058">https://doi.org/10.1002/wlb3.01058</a>.
</div>
<div id="ref-horswill2019" class="csl-entry" role="listitem">
Horswill, Cat, Holly K Kindsvater, Maria José Juan-Jordá, Nicholas K Dulvy, Marc Mangel, and Jason Matthiopoulos. 2019. <span>“Global Reconstruction of Life-History Strategies: A Case Study Using Tunas.”</span> <em>Journal of Applied Ecology</em> 56 (4): 855–65.
</div>
<div id="ref-horswill2021" class="csl-entry" role="listitem">
Horswill, Cat, Andrea Manica, Francis Daunt, Mark Newell, Sarah Wanless, Matthew Wood, and Jason Matthiopoulos. 2021. <span>“Improving Assessments of Data-Limited Populations Using Life-History Theory.”</span> <em>Journal of Applied Ecology</em> 58 (6): 1225–36.
</div>
<div id="ref-Israelsen2020" class="csl-entry" role="listitem">
Israelsen, M. F., L. F. Eriksen, P. F. Moa, B. R. Hagen, and E. B. Nilsen. 2020. <span>“Survival and Cause-Specific Mortality of Harvested Willow Ptarmigan (Lagopus Lagopus) in Central Norway.”</span> <em>Ecol Evol</em> 10 (20): 11144–54. <a href="https://doi.org/10.1002/ece3.6754">https://doi.org/10.1002/ece3.6754</a>.
</div>
<div id="ref-jakobsson2020" class="csl-entry" role="listitem">
Jakobsson, Simon, and Bård Pedersen. 2020. <span>“Naturindeks for Norge 2020. Tilstand Og Utvikling for Biologisk Mangfold.”</span> <em>NINA Rapport</em> 1886.
</div>
<div id="ref-jetz2019" class="csl-entry" role="listitem">
Jetz, Walter, Melodie A McGeoch, Robert Guralnick, Simon Ferrier, Jan Beck, Mark J Costello, Miguel Fernandez, et al. 2019. <span>“Essential Biodiversity Variables for Mapping and Monitoring Species Populations.”</span> <em>Nature Ecology &amp; Evolution</em> 3 (4): 539–51.
</div>
<div id="ref-johnston2023" class="csl-entry" role="listitem">
Johnston, Alison, Eleni Matechou, and Emily B Dennis. 2023. <span>“Outstanding Challenges and Future Directions for Biodiversity Monitoring Using Citizen Science Data.”</span> <em>Methods in Ecology and Evolution</em> 14 (1): 103–16.
</div>
<div id="ref-landau2021" class="csl-entry" role="listitem">
Landau, William Michael. 2021. <span>“The Targets r Package: A Dynamic Make-Like Function-Oriented Pipeline Toolkit for Reproducibility and High-Performance Computing.”</span> <em>Journal of Open Source Software</em> 6 (57): 2959. <a href="https://doi.org/10.21105/joss.02959">https://doi.org/10.21105/joss.02959</a>.
</div>
<div id="ref-Linden1988" class="csl-entry" role="listitem">
Linden, H. 1988. <span>“Latitudinal Gradients in Predator-Prey Interactions, Cyclicity and Synchronism in Voles and Small Game Populations in Finland.”</span> Journal Article. <em>Oikos</em> 52 (3): 341–49. <a href="<Go to ISI>://A1988N893200014 ">&lt;Go to ISI&gt;://A1988N893200014 </a>.
</div>
<div id="ref-morrison2022" class="csl-entry" role="listitem">
Morrison, Catriona A, Simon J Butler, Jacquie A Clark, Juan Arizaga, Oriol Baltà, Jaroslav Cepák, Arantza Leal Nebot, et al. 2022. <span>“Demographic Variation in Space and Time: Implications for Conservation Targeting.”</span> <em>Royal Society Open Science</em> 9 (3): 211671.
</div>
<div id="ref-nater2023" class="csl-entry" role="listitem">
Nater, Chloé R, Malcolm D Burgess, Peter Coffey, Bob Harris, Frank Lander, David Price, Mike Reed, and Robert A Robinson. 2023. <span>“Spatial Consistency in Drivers of Population Dynamics of a Declining Migratory Bird.”</span> <em>Journal of Animal Ecology</em> 92 (1): 97–111.
</div>
<div id="ref-nilsen2023" class="csl-entry" role="listitem">
Nilsen, Erlend B, and Chloé R Nater. 2023. <span>“An Integrated Distance Sampling Approach for Modelling Age-Structured Populations.”</span> <em>Manuscript</em>.
</div>
<div id="ref-nilsen2020" class="csl-entry" role="listitem">
Nilsen, Erlend B, and Lars Rød-Eriksen. 2020. <span>“Trender i St<span>ø</span>rrelsen p<span>å</span> Den Norske Lirypebestanden i Perioden 2009-2020. Analyser Basert p<span>å</span> Data Fra h<span>ø</span>nsefuglportalen.”</span> <em>NINA Rapport</em> 1869.
</div>
<div id="ref-pacifici2017" class="csl-entry" role="listitem">
Pacifici, Krishna, Brian J Reich, David AW Miller, Beth Gardner, Glenn Stauffer, Susheela Singh, Alexa McKerrow, and Jaime A Collazo. 2017. <span>“Integrating Multiple Data Sources in Species Distribution Modeling: A Framework for Data Fusion.”</span> <em>Ecology</em> 98 (3): 840–50.
</div>
<div id="ref-pereira2013" class="csl-entry" role="listitem">
Pereira, Henrique Miguel, Simon Ferrier, Michele Walters, Gary N Geller, Rob HG Jongman, Robert J Scholes, Michael William Bruford, et al. 2013. <span>“Essential Biodiversity Variables.”</span> <em>Science</em> 339 (6117): 277–78.
</div>
<div id="ref-proencca2017" class="csl-entry" role="listitem">
Proença, Vânia, Laura Jane Martin, Henrique Miguel Pereira, Miguel Fernandez, Louise McRae, Jayne Belnap, Monika Böhm, et al. 2017. <span>“Global Biodiversity Monitoring: From Data Sources to Essential Biodiversity Variables.”</span> <em>Biological Conservation</em> 213: 256–63.
</div>
<div id="ref-robinson2014" class="csl-entry" role="listitem">
Robinson, Robert A, Catriona A Morrison, and Stephen R Baillie. 2014. <span>“Integrating Demographic Data: Towards a Framework for Monitoring Wildlife Populations at Large Spatial Scales.”</span> <em>Methods in Ecology and Evolution</em> 5 (12): 1361–72.
</div>
<div id="ref-sandercock2005" class="csl-entry" role="listitem">
Sandercock, Brett K, Kathy Martin, and Susan J Hannon. 2005. <span>“Life History Strategies in Extreme Environments: Comparative Demography of Arctic and Alpine Ptarmigan.”</span> <em>Ecology</em> 86 (8): 2176–86.
</div>
<div id="ref-schmeller2018" class="csl-entry" role="listitem">
Schmeller, Dirk S, Lauren V Weatherdon, Adeline Loyau, Alberte Bondeau, Lluis Brotons, Neil Brummitt, Ilse R Geijzendorffer, et al. 2018. <span>“A Suite of Essential Biodiversity Variables for Detecting Critical Biodiversity Change.”</span> <em>Biological Reviews</em> 93 (1): 55–71.
</div>
<div id="ref-stearns1992" class="csl-entry" role="listitem">
Stearns, Stephen C. 1992. <em>The Evolution of Life Histories</em>. Vol. 249. Oxford university press Oxford.
</div>
<div id="ref-Steen1996" class="csl-entry" role="listitem">
Steen, H., and K. E. Erikstad. 1996. <span>“Sensivity of Willow Grouse Lagopus Lagopus Population Dynamics to Variations in Demographic Parameters.”</span> Journal Article. <em>Wildlife Biology</em> 2: 27–35.
</div>
<div id="ref-Steen1985" class="csl-entry" role="listitem">
Steen, J. B., H. Steen, N. C. Stenseth, S. Myrberget, and V. Marcström. 1988. <span>“Microtine Density and Weather as Predictors of Chick Production in Willow Ptarmigan, Lagopus Lagopus.”</span> Journal Article. <em>Oikos</em> 51 (3): 367–73.
</div>
<div id="ref-stevenson2021" class="csl-entry" role="listitem">
Stevenson, Simone L, Kate Watermeyer, Giovanni Caggiano, Elizabeth A Fulton, Simon Ferrier, and Emily Nicholson. 2021. <span>“Matching Biodiversity Indicators to Policy Needs.”</span> <em>Conservation Biology</em> 35 (2): 522–32.
</div>
<div id="ref-Storch2007" class="csl-entry" role="listitem">
Storch, Ilse. 2007. <span>“Conservation Status of Grouse Worldwide: An Update.”</span> Journal Article. <em>Wildlife Biology</em> 13 (sp1): 5–12. <a href="https://doi.org/10.2981/0909-6396(2007)13[5:csogwa]2.0.co;2">https://doi.org/10.2981/0909-6396(2007)13[5:csogwa]2.0.co;2</a>.
</div>
<div id="ref-valpine2017" class="csl-entry" role="listitem">
Valpine, Perry de, Daniel Turek, Christopher J Paciorek, Clifford Anderson-Bergman, Duncan Temple Lang, and Rastislav Bodik. 2017. <span>“Programming with Models: Writing Statistical Algorithms for General Model Structures with NIMBLE.”</span> <em>Journal of Computational and Graphical Statistics</em> 26 (2): 403–13.
</div>
<div id="ref-waldock2022" class="csl-entry" role="listitem">
Waldock, Conor, Rick D Stuart-Smith, Camille Albouy, William WL Cheung, Graham J Edgar, David Mouillot, Jerry Tjiputra, and Loı̈c Pellissier. 2022. <span>“A Quantitative Review of Abundance-Based Species Distribution Models.”</span> <em>Ecography</em> 2022 (1).
</div>
<div id="ref-williams2002" class="csl-entry" role="listitem">
Williams, Byron K, James D Nichols, and Michael J Conroy. 2002. <em>Analysis and Management of Animal Populations</em>. Academic Press.
</div>
</div></section></div></main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->



</body></html>